cluster/images/conformance/Makefile

```diff
@@ -54,9 +54,14 @@ endif
 	chmod a+rx ${TEMP_DIR}/e2e.test
 	chmod a+rx ${TEMP_DIR}/gorunner
 
-	cd ${TEMP_DIR} && sed -i.back "s|ARCH|${ARCH}|g; s|BASEIMAGE|${BASEIMAGE}|g" Dockerfile
+	cd ${TEMP_DIR} && sed -i.back "s|BASEIMAGE|${BASEIMAGE}|g" Dockerfile
 
-	docker build --pull -t ${REGISTRY}/conformance-${ARCH}:${VERSION} ${TEMP_DIR}
+	docker buildx build \
+		--platform linux/${ARCH} \
+		--load \
+		--pull \
```

Comment:
@saschagrunert are we not concerned about the base image caching issue that previously broke this when switching to buildx? (see https://github.com/kubernetes/kubernetes/pull/97982 for more info)

Reply:
Just a bit (?) :innocent: 

I tested it right now and it seems to work:
```
Step #2: +++ [0129 14:32:29] Building conformance image for arch: amd64
Step #2: time="2021-01-29T14:32:29Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:32:29Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:32:29Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:32:29Z" level=warning msg="quiet currently not implemented"
Step #2: #2 [internal] load .dockerignore
Step #2: #2 sha256:76eb46de3f768e219144139759aeb28c6d65e67e69291b36a468c17cb9f13b87
Step #2: #2 transferring context: 2B done
Step #2: #2 DONE 0.1s
Step #2: 
Step #2: #1 [internal] load build definition from Dockerfile
Step #2: #1 sha256:b64c6186c7103b261a4b1b927378f60ddb8466a3fe21e78e263cccb5072b603d
Step #2: #1 transferring dockerfile: 139B done
Step #2: #1 [internal] load build definition from Dockerfile
Step #2: #1 sha256:39f453febc6d07300b35c297cbd28986c852ebe1401a6948d606aff288fec629
Step #2: #1 transferring dockerfile: 157B done
Step #2: #1 DONE 0.1s
Step #2: 
Step #2: #2 [internal] load .dockerignore
Step #2: #2 sha256:0b99f72d97a3d8c2f67184deb8ea58e0616805787fabbb65ff68b3ad796b361d
Step #2: #2 transferring context: 2B done
Step #2: #2 DONE 0.1s
Step #2: #2 [internal] load .dockerignore
Step #2: #2 sha256:aab8d14512f79fce96910cbe32ce07662a4fc6a89f7ca78f99081bd1026edbf2
Step #2: #2 transferring context: 2B done
Step #2: #2 DONE 0.1s
Step #2: 
Step #2: #1 [internal] load build definition from Dockerfile
Step #2: #1 sha256:008c70a6d9a6a66505cdd2fdcbf778601db8d2863210b18fdb25f6d15bb03851
Step #2: #1 transferring dockerfile: 137B done
Step #2: #1 DONE 0.1s
Step #2: #2 [internal] load .dockerignore
Step #2: #2 sha256:707c2702ff6e8e67483e987589e0efa5303f3618bae364e4707dcf944bc55910
Step #2: #2 transferring context: 2B done
Step #2: #2 DONE 0.1s
Step #2: 
Step #2: #1 [internal] load build definition from Dockerfile
Step #2: #1 sha256:73d2f0c653b02445d1258bb68f2c1e788523e497786d34c026fcf427f12725ef
Step #2: #1 transferring dockerfile: 139B done
Step #2: #1 DONE 0.1s
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:a3f54e69457b5a439d9d985e797d04c956bfc2f8a656b77da4c09959298a2a27
Step #2: #1 DONE 0.1s
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:a3f54e69457b5a439d9d985e797d04c956bfc2f8a656b77da4c09959298a2a27
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:a3f54e69457b5a439d9d985e797d04c956bfc2f8a656b77da4c09959298a2a27
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0
Step #2: #3 sha256:db3e4cca567f67b6e044e2fda75dbbca017b7d7248a581844b26f990cbc7c35a
Step #2: #2 [internal] load build definition from Dockerfile
Step #2: #2 sha256:8c188eb7b75e14322b3b64fb569a9bffbea85b4318395bc8e44ca2d2adf82007
Step #2: #2 transferring dockerfile: 1.05kB done
Step #2: #2 DONE 0.0s
Step #2: 
Step #2: #1 [internal] load .dockerignore
Step #2: #1 sha256:409a3ac68fb8887366b8617201ca3664b8d0255355479a9d8c0c55dc4c2b9a82
Step #2: #1 transferring context: 2B done
Step #2: #1 DONE 0.0s
Step #2: 
Step #2: #3 [internal] load metadata for docker.io/library/debian:stable-slim
Step #2: #3 sha256:1f4135ad7b991da43d62098afc9b590afad696b89ebbcf75e55438bd2aee3a24
Step #2: #3 DONE 0.5s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb
Step #2: #4 sha256:23bd2b4ad845c658cbb2d84ef56e29bcb9ec8cdb1aab11329f9633ffb57b641f
Step #2: #4 resolve k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb done
Step #2: #3 DONE 0.5s
Step #2: #3 DONE 0.6s
Step #2: 
Step #2: #6 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #6 sha256:73c80ff3245c6ec8522e3ecfdfdbd7125a31a8674554baab9a3daa3f8e84b38e
Step #2: #6 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: #3 DONE 0.6s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:73c80ff3245c6ec8522e3ecfdfdbd7125a31a8674554baab9a3daa3f8e84b38e
Step #2: #4 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: #4 sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb 1.70kB / 1.70kB done
Step #2: #4 sha256:ef624f581144be5436389569385f667177cfce32044c42c09264f5c46d83378e 2.68kB / 2.68kB done
Step #2: #4 sha256:80b8c7bdf1f12d199fce1b1174817ef49b2b8a93b00ddc25c2ddb2b21d13eca1 0B / 1.97MB 0.1s
Step #2: #4 sha256:b4b8b1e0d4617011dd03f20b804cc2e50bf48bafc36b1c8c7bd23fd44bfd641e 1.57kB / 1.57kB done
Step #2: #4 sha256:1813d21adc01d6699dcebdaf72ab66d2839755b536cedba34251fb39a9e9f02b 0B / 21.09MB 0.1s
Step #2: #4 sha256:98435e81eab0fa9aaec073adcfc8eea85c7c033f002fbb5bcf9ac1c101cd6280 0B / 15.03MB 0.1s
Step #2: #3 DONE 0.7s
Step #2: 
Step #2: #6 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #6 sha256:73c80ff3245c6ec8522e3ecfdfdbd7125a31a8674554baab9a3daa3f8e84b38e
Step #2: #6 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: 
Step #2: #12 [1/8] FROM docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350
Step #2: #12 sha256:8818c1c8b9c17cdb62fbb16677b1905594f204bbdf4593e8137a9e6fa8b895b3
Step #2: #12 resolve docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350 done
Step #2: #6 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #6 sha256:66cdf0be9ba12e3183c43664f9f02b14e2642b19a3d82784d15f56d01e521017 737B / 737B done
Step #2: #6 sha256:3c312f5a486cac7c2d3d134d72de97708a40c60fd642c9e1573c89af77824fff 1.41kB / 1.41kB done
Step #2: #4 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #4 sha256:66cdf0be9ba12e3183c43664f9f02b14e2642b19a3d82784d15f56d01e521017 737B / 737B done
Step #2: #4 sha256:3c312f5a486cac7c2d3d134d72de97708a40c60fd642c9e1573c89af77824fff 1.41kB / 1.41kB done
Step #2: #6 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #6 sha256:66cdf0be9ba12e3183c43664f9f02b14e2642b19a3d82784d15f56d01e521017 737B / 737B done
Step #2: #6 sha256:3c312f5a486cac7c2d3d134d72de97708a40c60fd642c9e1573c89af77824fff 1.41kB / 1.41kB done
Step #2: #12 sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350 1.85kB / 1.85kB done
Step #2: #12 sha256:f8d3f188cb883482e7dd4edf92f03c130094e36b8e35315e319f27b0e51394dd 529B / 529B done
Step #2: #12 sha256:86154f0d62745193ef833869c3b8e1fde5b28d0265df0bcc74ba57a5530d56a2 1.46kB / 1.46kB done
Step #2: #4 sha256:80b8c7bdf1f12d199fce1b1174817ef49b2b8a93b00ddc25c2ddb2b21d13eca1 261.00kB / 1.97MB 0.3s
Step #2: #4 sha256:98435e81eab0fa9aaec073adcfc8eea85c7c033f002fbb5bcf9ac1c101cd6280 6.34MB / 15.03MB 0.3s
Step #2: #4 sha256:80b8c7bdf1f12d199fce1b1174817ef49b2b8a93b00ddc25c2ddb2b21d13eca1 1.97MB / 1.97MB 0.3s done
Step #2: #4 sha256:1813d21adc01d6699dcebdaf72ab66d2839755b536cedba34251fb39a9e9f02b 8.39MB / 21.09MB 0.6s
Step #2: #4 sha256:98435e81eab0fa9aaec073adcfc8eea85c7c033f002fbb5bcf9ac1c101cd6280 15.03MB / 15.03MB 0.6s
Step #2: #4 sha256:0a60cc3d3112991db6b7cd622133bf5bef3b07313d8701e2d3cf2dd5fb29ed5c 1.03kB / 1.03kB 0.6s
Step #2: #4 ...
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:df445235c0dd9299b5aa646efca964aead8b1261349cc82291db41965e263734
Step #2: #5 transferring context: 39.53MB 0.5s done
Step #2: #5 DONE 0.7s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb
Step #2: #4 sha256:23bd2b4ad845c658cbb2d84ef56e29bcb9ec8cdb1aab11329f9633ffb57b641f
Step #2: #6 ...
Step #2: 
Step #2: #4 [internal] load build context
Step #2: #4 sha256:cd5cdc634d657b053cd1c59c1a5397778e35e7671b273726a987c7427ac27aed
Step #2: #4 transferring context: 42.95MB 0.5s done
Step #2: #4 DONE 1.1s
Step #2: #4 extracting sha256:1813d21adc01d6699dcebdaf72ab66d2839755b536cedba34251fb39a9e9f02b
Step #2: 
Step #2: #6 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #6 sha256:73c80ff3245c6ec8522e3ecfdfdbd7125a31a8674554baab9a3daa3f8e84b38e
Step #2: #6 sha256:d3067e65b05aa5d68b61fb18c5796eed35b66883f6b250b8979d04f1591e7beb 0B / 649.10kB 1.3s
Step #2: #6 sha256:d3067e65b05aa5d68b61fb18c5796eed35b66883f6b250b8979d04f1591e7beb 0B / 649.10kB 1.3s
Step #2: #4 sha256:d3067e65b05aa5d68b61fb18c5796eed35b66883f6b250b8979d04f1591e7beb 0B / 649.10kB 1.3s
Step #2: #4 sha256:1813d21adc01d6699dcebdaf72ab66d2839755b536cedba34251fb39a9e9f02b 21.09MB / 21.09MB 1.1s done
Step #2: #4 sha256:98435e81eab0fa9aaec073adcfc8eea85c7c033f002fbb5bcf9ac1c101cd6280 15.03MB / 15.03MB 0.7s done
Step #2: #4 sha256:0a60cc3d3112991db6b7cd622133bf5bef3b07313d8701e2d3cf2dd5fb29ed5c 1.03kB / 1.03kB 0.9s done
Step #2: #4 sha256:c9b12ef85413b68f3ca3beed78b31f3c10a607969062dec45d120e05817db201 545B / 545B 1.4s
Step #2: #4 sha256:061309079d300544d0e9319170ba8f6d45fcd013929d0d12ac5bb40dc4853c86 507B / 507B 1.4s
Step #2: #4 extracting sha256:d3067e65b05aa5d68b61fb18c5796eed35b66883f6b250b8979d04f1591e7beb
Step #2: #6 extracting sha256:d3067e65b05aa5d68b61fb18c5796eed35b66883f6b250b8979d04f1591e7beb 0.1s
Step #2: #6 extracting sha256:d3067e65b05aa5d68b61fb18c5796eed35b66883f6b250b8979d04f1591e7beb 0.1s
Step #2: #4 extracting sha256:d3067e65b05aa5d68b61fb18c5796eed35b66883f6b250b8979d04f1591e7beb 0.2s done
Step #2: #12 sha256:d94a301dc4fcea3e371c002722f8d76ccaa8f71b352d735d508e0b829ad63be0 0B / 27.11MB 1.7s
Step #2: #6 sha256:d3067e65b05aa5d68b61fb18c5796eed35b66883f6b250b8979d04f1591e7beb 649.10kB / 649.10kB 1.6s done
Step #2: #6 extracting sha256:d3067e65b05aa5d68b61fb18c5796eed35b66883f6b250b8979d04f1591e7beb 0.2s done
Step #2: #6 sha256:592fdd98d65be3cad5ff2f359c59c592077a9105fc1c46b4f1b9ff6ea12b9954 0B / 723.85kB 1.8s
Step #2: #6 sha256:d3067e65b05aa5d68b61fb18c5796eed35b66883f6b250b8979d04f1591e7beb 649.10kB / 649.10kB 1.6s done
Step #2: #6 extracting sha256:d3067e65b05aa5d68b61fb18c5796eed35b66883f6b250b8979d04f1591e7beb 0.2s done
Step #2: #6 sha256:592fdd98d65be3cad5ff2f359c59c592077a9105fc1c46b4f1b9ff6ea12b9954 0B / 723.85kB 1.8s
Step #2: #4 sha256:c9b12ef85413b68f3ca3beed78b31f3c10a607969062dec45d120e05817db201 545B / 545B 1.5s done
Step #2: #4 sha256:061309079d300544d0e9319170ba8f6d45fcd013929d0d12ac5bb40dc4853c86 507B / 507B 1.5s done
Step #2: #4 sha256:d3067e65b05aa5d68b61fb18c5796eed35b66883f6b250b8979d04f1591e7beb 649.10kB / 649.10kB 1.6s done
Step #2: #4 sha256:592fdd98d65be3cad5ff2f359c59c592077a9105fc1c46b4f1b9ff6ea12b9954 0B / 723.85kB 1.9s
Step #2: #12 sha256:d94a301dc4fcea3e371c002722f8d76ccaa8f71b352d735d508e0b829ad63be0 8.39MB / 27.11MB 2.0s
Step #2: #6 sha256:592fdd98d65be3cad5ff2f359c59c592077a9105fc1c46b4f1b9ff6ea12b9954 723.85kB / 723.85kB 2.1s
Step #2: #4 sha256:592fdd98d65be3cad5ff2f359c59c592077a9105fc1c46b4f1b9ff6ea12b9954 723.85kB / 723.85kB 2.1s
Step #2: #6 sha256:592fdd98d65be3cad5ff2f359c59c592077a9105fc1c46b4f1b9ff6ea12b9954 723.85kB / 723.85kB 2.1s
Step #2: #6 ...
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:5fe1c6348862e77897a307feaf0cc2ff3e69148f22d3b0315b73c4c5f606b02c
Step #2: #5 transferring context: 118.19MB 1.5s done
Step #2: #5 DONE 2.3s
Step #2: 
Step #2: #6 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #6 sha256:73c80ff3245c6ec8522e3ecfdfdbd7125a31a8674554baab9a3daa3f8e84b38e
Step #2: #6 sha256:592fdd98d65be3cad5ff2f359c59c592077a9105fc1c46b4f1b9ff6ea12b9954 723.85kB / 723.85kB 2.2s done
Step #2: #4 sha256:592fdd98d65be3cad5ff2f359c59c592077a9105fc1c46b4f1b9ff6ea12b9954 723.85kB / 723.85kB 2.2s done
Step #2: #6 sha256:592fdd98d65be3cad5ff2f359c59c592077a9105fc1c46b4f1b9ff6ea12b9954 723.85kB / 723.85kB 2.2s done
Step #2: #6 extracting sha256:592fdd98d65be3cad5ff2f359c59c592077a9105fc1c46b4f1b9ff6ea12b9954 0.1s done
Step #2: #4 extracting sha256:592fdd98d65be3cad5ff2f359c59c592077a9105fc1c46b4f1b9ff6ea12b9954 0.1s done
Step #2: #6 extracting sha256:592fdd98d65be3cad5ff2f359c59c592077a9105fc1c46b4f1b9ff6ea12b9954 0.1s done
Step #2: #4 extracting sha256:1813d21adc01d6699dcebdaf72ab66d2839755b536cedba34251fb39a9e9f02b 1.3s done
Step #2: #4 ...
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:3b1123f5b3dc1ab39ca4c8a1b55321344e5e2c782423a8f65a5261f41a91c327
Step #2: #5 transferring context: 112.80MB 1.9s done
Step #2: #5 DONE 2.6s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:73c80ff3245c6ec8522e3ecfdfdbd7125a31a8674554baab9a3daa3f8e84b38e
Step #2: #12 sha256:d94a301dc4fcea3e371c002722f8d76ccaa8f71b352d735d508e0b829ad63be0 27.11MB / 27.11MB 3.0s
Step #2: #12 ...
Step #2: 
Step #2: #4 [internal] load build context
Step #2: #4 sha256:4ed7652fe9788fab7b01e750d4315fe045350b339b0e7823507ed9e8a643d0e4
Step #2: #4 transferring context: 174.00MB 2.2s done
Step #2: #4 DONE 3.1s
Step #2: 
Step #2: #12 [1/8] FROM docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350
Step #2: #12 sha256:8818c1c8b9c17cdb62fbb16677b1905594f204bbdf4593e8137a9e6fa8b895b3
Step #2: #12 sha256:d94a301dc4fcea3e371c002722f8d76ccaa8f71b352d735d508e0b829ad63be0 27.11MB / 27.11MB 3.4s done
Step #2: #4 extracting sha256:98435e81eab0fa9aaec073adcfc8eea85c7c033f002fbb5bcf9ac1c101cd6280
Step #2: #12 extracting sha256:d94a301dc4fcea3e371c002722f8d76ccaa8f71b352d735d508e0b829ad63be0
Step #2: #4 extracting sha256:98435e81eab0fa9aaec073adcfc8eea85c7c033f002fbb5bcf9ac1c101cd6280 0.3s done
Step #2: #4 DONE 4.0s
Step #2: #6 DONE 4.0s
Step #2: #6 DONE 4.0s
Step #2: 
Step #2: #6 [2/2] COPY kube-controller-manager /usr/local/bin/kube-controller-manager
Step #2: #6 sha256:f50eb126670359b486d35df0f5732150668d18b3bfd67172b39c033d2598fc23
Step #2: 
Step #2: #4 [2/2] COPY kube-apiserver /usr/local/bin/kube-apiserver
Step #2: 
Step #2: #4 sha256:12527396284567fd728b4dc7939b210ab969db77ec4a208342b307fb527da209
Step #2: #5 [2/2] COPY kube-scheduler /usr/local/bin/kube-scheduler
Step #2: #5 sha256:253d9c9fd7db7590a8a4b50a9cf3eabcc9b06eae8ce73207b3f8d35004c93068
Step #2: #4 extracting sha256:80b8c7bdf1f12d199fce1b1174817ef49b2b8a93b00ddc25c2ddb2b21d13eca1
Step #2: #4 extracting sha256:80b8c7bdf1f12d199fce1b1174817ef49b2b8a93b00ddc25c2ddb2b21d13eca1 0.1s done
Step #2: #12 extracting sha256:d94a301dc4fcea3e371c002722f8d76ccaa8f71b352d735d508e0b829ad63be0 1.2s done
Step #2: #4 extracting sha256:0a60cc3d3112991db6b7cd622133bf5bef3b07313d8701e2d3cf2dd5fb29ed5c done
Step #2: #12 DONE 6.2s
Step #2: #4 extracting sha256:061309079d300544d0e9319170ba8f6d45fcd013929d0d12ac5bb40dc4853c86
Step #2: 
Step #2: #5 [2/8] COPY ginkgo /usr/local/bin/
Step #2: #5 sha256:93de967251d50c373fb88387b6360bc6313c0e0adfc953244772d9ff7f1d0fcb
Step #2: #4 extracting sha256:061309079d300544d0e9319170ba8f6d45fcd013929d0d12ac5bb40dc4853c86 done
Step #2: #4 extracting sha256:c9b12ef85413b68f3ca3beed78b31f3c10a607969062dec45d120e05817db201
Step #2: #4 extracting sha256:c9b12ef85413b68f3ca3beed78b31f3c10a607969062dec45d120e05817db201 done
Step #2: #4 DONE 3.4s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #5 DONE 3.4s
Step #2: #6 DONE 3.4s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #4 DONE 7.6s
Step #2: 
Step #2: #6 [2/2] COPY kube-proxy /usr/local/bin/kube-proxy
Step #2: #6 sha256:842955e129907285c87cca11345f6b94140bfb0d248225b51aa2e0b330c7e0a1
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #5 DONE 1.1s
Step #2: 
Step #2: #6 [3/8] COPY e2e.test /usr/local/bin/
Step #2: #6 sha256:e7711ca8678d7cadf10a8009c972435efc6976a4cd6b9fe77ec9bce393af2728
Step #2: #6 DONE 0.7s
Step #2: #7 exporting layers 0.7s done
Step #2: #7 writing image sha256:a602021bac802bf6dc6ca93e05cac15f3aed221a6d26efff6faae9590df52cf6 0.0s done
Step #2: #7 naming to k8s.gcr.io/kube-scheduler-amd64:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.7s
Step #2: #7 exporting layers 0.7s done
Step #2: #7 exporting layers 0.7s done
Step #2: #7 writing image sha256:dda3898b5f5a0ef9f9800a0d7959582121c54b8c55a3380975e5dd4b747abbc2 done
Step #2: #7 naming to k8s.gcr.io/kube-apiserver-amd64:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.7s
Step #2: #7 writing image sha256:31aeb1ab92f656a2525508fdc448dabbd6771a88edee9a7ae57fba0061f47d0e done
Step #2: #7 naming to k8s.gcr.io/kube-controller-manager-amd64:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.7s
Step #2: #6 DONE 0.7s
Step #2: 
Step #2: #7 [4/8] COPY kubectl /usr/local/bin/
Step #2: #7 sha256:013e5417864f9ac84350d59d69c0f423c43cd18fd7813a42c6350decec4f929d
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #7 DONE 0.3s
Step #2: #7 exporting layers 0.3s done
Step #2: #7 writing image sha256:47093ee8ade48a7bb94dbb8da4727d66f032a1bde578f5e223b472168bb0894b done
Step #2: #7 naming to k8s.gcr.io/kube-proxy-amd64:v1.21.0-alpha.3 0.0s done
Step #2: #7 DONE 0.4s
Step #2: 
Step #2: #8 [5/8] COPY run_e2e.sh /run_e2e.sh
Step #2: #8 sha256:a3865d52036eaf5f94866dd4b1f35951cbf137eb44d062bcc2f586ef61022bd4
Step #2: +++ [0129 14:32:40] Deleting docker image k8s.gcr.io/kube-scheduler-amd64:v1.21.0-alpha.3
Step #2: #8 DONE 1.5s
Step #2: 
Step #2: #9 [6/8] COPY gorunner /gorunner
Step #2: #9 sha256:ad8dc34440ed1e55de7cafeff4978c2697c1f156ba2d98239c96bbed30cb68b2
Step #2: +++ [0129 14:32:40] Deleting docker image k8s.gcr.io/kube-apiserver-amd64:v1.21.0-alpha.3
Step #2: +++ [0129 14:32:40] Deleting docker image k8s.gcr.io/kube-controller-manager-amd64:v1.21.0-alpha.3
Step #2: +++ [0129 14:32:40] Deleting docker image k8s.gcr.io/kube-proxy-amd64:v1.21.0-alpha.3
Step #2: #9 DONE 1.7s
Step #2: 
Step #2: #10 [7/8] COPY cluster /kubernetes/cluster
Step #2: #10 sha256:76bf9c08de12acdd462f0ebb4fbb6bd81a59745fffe0d659d07b2108d701c239
Step #2: #10 DONE 0.1s
Step #2: 
Step #2: #11 [8/8] WORKDIR /usr/local/bin
Step #2: #11 sha256:aad8e63779a00dcf4bd6bfa24516b366ee1efdaced6988e1682c573e308478a6
Step #2: #11 DONE 0.0s
Step #2: 
Step #2: #13 exporting to image
Step #2: #13 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #13 exporting layers
Step #2: #13 exporting layers 0.7s done
Step #2: #13 writing image sha256:cae9f6a629cc80889efeecce304214fedaf37d36519212d82e3ee5c9d444f6ab done
Step #2: #13 naming to k8s.gcr.io/conformance-amd64:v1.21.0-alpha.3 done
Step #2: #13 DONE 0.7s
Step #2: +++ [0129 14:32:46] Building tarball: node linux-arm
Step #2: +++ [0129 14:32:46] Deleting conformance image k8s.gcr.io/conformance-amd64:v1.21.0-alpha.3
Step #2: +++ [0129 14:32:46] Docker builds done
Step #2: +++ [0129 14:32:46] Building images: linux-arm
Step #2: +++ [0129 14:32:46] Starting docker build for image: kube-apiserver-arm
Step #2: +++ [0129 14:32:46] Starting docker build for image: kube-controller-manager-arm
Step #2: +++ [0129 14:32:46] Starting docker build for image: kube-scheduler-arm
Step #2: +++ [0129 14:32:46] Starting docker build for image: kube-proxy-arm
Step #2: +++ [0129 14:32:46] Building conformance image for arch: arm
Step #2: time="2021-01-29T14:32:46Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:32:46Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:32:46Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:32:46Z" level=warning msg="quiet currently not implemented"
Step #2: #1 [internal] load build definition from Dockerfile
Step #2: #1 sha256:f1557fbb60ea741e75deb1a6bc6b77a5ab126f678811678741090d71709fe8f2
Step #2: #1 [internal] load build definition from Dockerfile
Step #2: #1 sha256:1f0e4aaf586bea267d138e1eeda437e708e29a27ee9354769519299b0c476458
Step #2: #1 transferring dockerfile: 139B done
Step #2: #1 DONE 0.1s
Step #2: 
Step #2: #2 [internal] load .dockerignore
Step #2: #2 sha256:fdbc2f7f3a4bfbaf0a3eda71bc722cb3a6ebe278b6a05d1fccc7759a17305178
Step #2: #2 transferring context: 2B done
Step #2: #2 DONE 0.1s
Step #2: #1 [internal] load .dockerignore
Step #2: #1 sha256:034bcf9f296c3c750244cae506bbd528887968cdc7188137d4b6a2ea4cc06c5c
Step #2: #1 transferring context: 2B done
Step #2: #1 DONE 0.1s
Step #2: 
Step #2: #2 [internal] load build definition from Dockerfile
Step #2: #2 sha256:6e0323b5bb9f4630693f66167dc7f4e6039a596f7166caa2e0a373e17271517b
Step #2: #2 transferring dockerfile: 137B done
Step #2: #2 DONE 0.1s
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0
Step #2: #3 sha256:24d0af2ee0197c409d1ef1ced7f6e12cd4a724b987a9107bb734c9e6b9d3865e
Step #2: #1 [internal] load .dockerignore
Step #2: #1 sha256:07c78540e000d1e87c79997b97b4e2e5a26c9f3419f3faff6afd900fe2a5c9ce
Step #2: #1 transferring context: 2B done
Step #2: #1 DONE 0.1s
Step #2: 
Step #2: #2 [internal] load build definition from Dockerfile
Step #2: #2 sha256:bed35784820052a617a454c42ee2f42e93202b96f49ea278bed8c765f9d9a5ac
Step #2: #2 transferring dockerfile: 157B done
Step #2: #1 transferring dockerfile: 139B done
Step #2: #1 DONE 0.0s
Step #2: 
Step #2: #2 [internal] load .dockerignore
Step #2: #2 sha256:bd68f36f24436147ee6629bc9cab1ed638953015741c7d88ff422d388f5853b6
Step #2: #2 transferring context: 2B done
Step #2: #2 DONE 0.0s
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:66f9ac794a42cea03fbd8ea9b2855d7c67e93397487f2cb8a9547e25e202e7cf
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:66f9ac794a42cea03fbd8ea9b2855d7c67e93397487f2cb8a9547e25e202e7cf
Step #2: #2 DONE 0.1s
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:66f9ac794a42cea03fbd8ea9b2855d7c67e93397487f2cb8a9547e25e202e7cf
Step #2: #1 [internal] load .dockerignore
Step #2: #1 sha256:bede130b84bc059c01ede226a531244321dd4d0786f7c9285dad7afc59a3e6e6
Step #2: #1 transferring context: 2B done
Step #2: #1 DONE 0.0s
Step #2: 
Step #2: #2 [internal] load build definition from Dockerfile
Step #2: #2 sha256:223e9d628e357ec2e4f8b8b655532d40b3d06f618c6f27c9412aa63f3e48b24f
Step #2: #2 transferring dockerfile: 1.05kB done
Step #2: #2 DONE 0.0s
Step #2: 
Step #2: #3 [internal] load metadata for docker.io/library/debian:stable-slim
Step #2: #3 sha256:47133cdd9944b5cc1c4bd8bf62b4c349242098f093d758451bb3301b5eac8996
Step #2: #3 DONE 0.4s
Step #2: #3 DONE 0.4s
Step #2: #3 DONE 0.5s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:e3e255809c4e7de3811274491cae72337806d2937188bdd3a4ecbc00049b996e
Step #2: #4 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:f09b1ccc2cff4c7a968696c44b834d90da91198a518ecaa6c0500dd5f9ee01b1
Step #2: 
Step #2: #5 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #5 sha256:e3e255809c4e7de3811274491cae72337806d2937188bdd3a4ecbc00049b996e
Step #2: #5 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: #3 DONE 0.5s
Step #2: 
Step #2: #6 [1/2] FROM k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb
Step #2: #6 sha256:bd9b624a366bcfbc1d5a7e5435ce5912f11a636f693d295ce38b0df2bf6373ac
Step #2: #6 resolve k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb done
Step #2: #4 sha256:73d168d8065565ff9a9509651efc5644907a5f4638f9ffca73a14136f4e6dfb0 0B / 649.10kB 0.1s
Step #2: #4 sha256:7d833bbdf8f4c1a90283dcc844cd60b3a188eef472c554557bd59767eeb4f65d 0B / 685.25kB 0.1s
Step #2: #4 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #4 sha256:61eefb4de21f946c1d4bc8e941f17a2a77c38f66c8b4a07d6da6d1e8918d47c4 737B / 737B done
Step #2: #4 sha256:42c44d104864bf4333bebebbad320fd1283d09c53ae86fa46f778e1af98939ad 1.43kB / 1.43kB done
Step #2: #5 sha256:73d168d8065565ff9a9509651efc5644907a5f4638f9ffca73a14136f4e6dfb0 0B / 649.10kB 0.1s
Step #2: #5 sha256:7d833bbdf8f4c1a90283dcc844cd60b3a188eef472c554557bd59767eeb4f65d 0B / 685.25kB 0.1s
Step #2: #5 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #5 sha256:61eefb4de21f946c1d4bc8e941f17a2a77c38f66c8b4a07d6da6d1e8918d47c4 737B / 737B done
Step #2: #5 sha256:42c44d104864bf4333bebebbad320fd1283d09c53ae86fa46f778e1af98939ad 1.43kB / 1.43kB done
Step #2: #6 sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb 1.70kB / 1.70kB done
Step #2: #6 sha256:09f79b3a00268705a8f8462f1528fed536e204905359f21e9965f08dd306c60a 1.57kB / 1.57kB done
Step #2: #6 sha256:74596c549d969003938911dcac96e9af15c4dccb9e8edd8dbd1a71af1b96559a 2.69kB / 2.69kB done
Step #2: #6 sha256:920dd5dc200ba523d4d8395eab01fc977e4507a87765a66b59552224c6a4c22f 0B / 20.78MB 0.1s
Step #2: #4 extracting sha256:73d168d8065565ff9a9509651efc5644907a5f4638f9ffca73a14136f4e6dfb0
Step #2: #3 DONE 0.5s
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:76abff18b346f454db296a85c17a7e6e92c619a463b38b759bcafd4161111785
Step #2: #5 DONE 0.0s
Step #2: 
Step #2: #4 [1/8] FROM docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350
Step #2: #4 sha256:c3005ada81c737c561a49ebcd38bc23df2db163a25aa637cb52527909f3d0e30
Step #2: #4 resolve docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350 done
Step #2: #4 DONE 0.0s
Step #2: #5 ...
Step #2: 
Step #2: #6 [internal] load build context
Step #2: #6 sha256:41d7784f60f434fadd7bb3e2dc9efa8e3430fb23c0666850e5b343bd25b90c36
Step #2: #6 transferring context: 35.53MB 0.4s done
Step #2: #6 DONE 0.4s
Step #2: 
Step #2: #5 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #5 sha256:e3e255809c4e7de3811274491cae72337806d2937188bdd3a4ecbc00049b996e
Step #2: #5 sha256:73d168d8065565ff9a9509651efc5644907a5f4638f9ffca73a14136f4e6dfb0 649.10kB / 649.10kB 0.2s done
Step #2: #5 sha256:7d833bbdf8f4c1a90283dcc844cd60b3a188eef472c554557bd59767eeb4f65d 69.63kB / 685.25kB 0.3s
Step #2: #5 extracting sha256:73d168d8065565ff9a9509651efc5644907a5f4638f9ffca73a14136f4e6dfb0
Step #2: #4 sha256:73d168d8065565ff9a9509651efc5644907a5f4638f9ffca73a14136f4e6dfb0 649.10kB / 649.10kB 0.2s done
Step #2: #4 sha256:7d833bbdf8f4c1a90283dcc844cd60b3a188eef472c554557bd59767eeb4f65d 69.63kB / 685.25kB 0.3s
Step #2: #6 ...
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:1e9af8591f299725706fb3895e69df9e6d1ff944247356124d69b9217ed08081
Step #2: #5 transferring context: 32.71MB 0.4s done
Step #2: #5 DONE 0.4s
Step #2: 
Step #2: #6 [1/2] FROM k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb
Step #2: #6 sha256:bd9b624a366bcfbc1d5a7e5435ce5912f11a636f693d295ce38b0df2bf6373ac
Step #2: #6 sha256:538547ed7102bd8629ea842d87f0345076c84bee08cf0fa41dad3f1ddaf5f4a8 0B / 15.04MB 0.2s
Step #2: #6 sha256:165d8763efcc37f35d190368bd2d4522e21584f8e9850f25d3de9ab04dbdd978 0B / 1.56MB 0.2s
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:76abff18b346f454db296a85c17a7e6e92c619a463b38b759bcafd4161111785
Step #2: #5 sha256:7d833bbdf8f4c1a90283dcc844cd60b3a188eef472c554557bd59767eeb4f65d 685.25kB / 685.25kB 0.3s done
Step #2: #5 extracting sha256:73d168d8065565ff9a9509651efc5644907a5f4638f9ffca73a14136f4e6dfb0 0.2s done
Step #2: #4 sha256:7d833bbdf8f4c1a90283dcc844cd60b3a188eef472c554557bd59767eeb4f65d 685.25kB / 685.25kB 0.3s done
Step #2: #4 extracting sha256:73d168d8065565ff9a9509651efc5644907a5f4638f9ffca73a14136f4e6dfb0 0.2s done
Step #2: #6 sha256:538547ed7102bd8629ea842d87f0345076c84bee08cf0fa41dad3f1ddaf5f4a8 2.01MB / 15.04MB 0.5s
Step #2: #4 extracting sha256:7d833bbdf8f4c1a90283dcc844cd60b3a188eef472c554557bd59767eeb4f65d
Step #2: #5 extracting sha256:7d833bbdf8f4c1a90283dcc844cd60b3a188eef472c554557bd59767eeb4f65d
Step #2: #6 sha256:920dd5dc200ba523d4d8395eab01fc977e4507a87765a66b59552224c6a4c22f 8.39MB / 20.78MB 0.5s
Step #2: #6 sha256:538547ed7102bd8629ea842d87f0345076c84bee08cf0fa41dad3f1ddaf5f4a8 8.39MB / 15.04MB 0.5s
Step #2: #4 extracting sha256:7d833bbdf8f4c1a90283dcc844cd60b3a188eef472c554557bd59767eeb4f65d 0.1s done
Step #2: #4 DONE 0.8s
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:741f14efc2e0e055e4aee762a267e4558e9a3e2851ec19a801277c7abe2ce542
Step #2: #5 ...
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:e3e255809c4e7de3811274491cae72337806d2937188bdd3a4ecbc00049b996e
Step #2: #4 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: #4 sha256:73d168d8065565ff9a9509651efc5644907a5f4638f9ffca73a14136f4e6dfb0 649.10kB / 649.10kB 0.2s done
Step #2: #4 sha256:7d833bbdf8f4c1a90283dcc844cd60b3a188eef472c554557bd59767eeb4f65d 685.25kB / 685.25kB 0.3s done
Step #2: #4 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #4 sha256:61eefb4de21f946c1d4bc8e941f17a2a77c38f66c8b4a07d6da6d1e8918d47c4 737B / 737B done
Step #2: #4 sha256:42c44d104864bf4333bebebbad320fd1283d09c53ae86fa46f778e1af98939ad 1.43kB / 1.43kB done
Step #2: #4 extracting sha256:73d168d8065565ff9a9509651efc5644907a5f4638f9ffca73a14136f4e6dfb0 0.2s done
Step #2: #4 extracting sha256:7d833bbdf8f4c1a90283dcc844cd60b3a188eef472c554557bd59767eeb4f65d 0.1s done
Step #2: #4 DONE 0.8s
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:f09b1ccc2cff4c7a968696c44b834d90da91198a518ecaa6c0500dd5f9ee01b1
Step #2: #5 extracting sha256:7d833bbdf8f4c1a90283dcc844cd60b3a188eef472c554557bd59767eeb4f65d 0.1s done
Step #2: #5 DONE 0.8s
Step #2: 
Step #2: #4 [2/2] COPY kube-scheduler /usr/local/bin/kube-scheduler
Step #2: #4 sha256:c59583caf6c43bdb7333b3caa04830cf84e42b59948ea106856ad6a0a78a24c2
Step #2: #6 sha256:920dd5dc200ba523d4d8395eab01fc977e4507a87765a66b59552224c6a4c22f 9.49MB / 20.78MB 0.7s
Step #2: #6 sha256:538547ed7102bd8629ea842d87f0345076c84bee08cf0fa41dad3f1ddaf5f4a8 15.04MB / 15.04MB 0.7s done
Step #2: #6 sha256:165d8763efcc37f35d190368bd2d4522e21584f8e9850f25d3de9ab04dbdd978 1.56MB / 1.56MB 0.6s done
Step #2: #6 sha256:c8381f480814de4b1bb2d189befa5a3318ad99fe19b92882680ee689b35b3e17 0B / 520B 0.7s
Step #2: #6 sha256:0044f1d60d03253b06d22435ba89506a033d769d77b70d0547570ce37db2015a 0B / 1.03kB 0.7s
Step #2: #6 sha256:920dd5dc200ba523d4d8395eab01fc977e4507a87765a66b59552224c6a4c22f 16.78MB / 20.78MB 0.8s
Step #2: #6 sha256:0044f1d60d03253b06d22435ba89506a033d769d77b70d0547570ce37db2015a 1.03kB / 1.03kB 0.8s
Step #2: #6 sha256:c8381f480814de4b1bb2d189befa5a3318ad99fe19b92882680ee689b35b3e17 520B / 520B 0.9s
Step #2: #5 transferring context: 95.63MB 1.2s done
Step #2: #5 transferring context: 91.24MB 1.2s done
Step #2: #6 sha256:920dd5dc200ba523d4d8395eab01fc977e4507a87765a66b59552224c6a4c22f 20.78MB / 20.78MB 1.3s
Step #2: #6 sha256:c8381f480814de4b1bb2d189befa5a3318ad99fe19b92882680ee689b35b3e17 520B / 520B 1.1s done
Step #2: #6 sha256:0044f1d60d03253b06d22435ba89506a033d769d77b70d0547570ce37db2015a 1.03kB / 1.03kB 1.1s done
Step #2: #6 sha256:7d49b9bee91c70bbb99661bd85d55fa30b7ade453a7c349eb7d0cffa60056160 0B / 548B 1.3s
Step #2: #5 DONE 1.5s
Step #2: 
Step #2: #6 [2/2] COPY kube-controller-manager /usr/local/bin/kube-controller-manager
Step #2: #6 sha256:c0caceb823e59eefa3c005f00d859b65042308c02632a881dcf45d83833374e6
Step #2: #6 sha256:7d49b9bee91c70bbb99661bd85d55fa30b7ade453a7c349eb7d0cffa60056160 548B / 548B 1.4s
Step #2: #5 DONE 1.7s
Step #2: #6 sha256:920dd5dc200ba523d4d8395eab01fc977e4507a87765a66b59552224c6a4c22f 20.78MB / 20.78MB 1.5s done
Step #2: #6 sha256:7d49b9bee91c70bbb99661bd85d55fa30b7ade453a7c349eb7d0cffa60056160 548B / 548B 1.5s done
Step #2: #6 extracting sha256:920dd5dc200ba523d4d8395eab01fc977e4507a87765a66b59552224c6a4c22f
Step #2: 
Step #2: #6 [2/2] COPY kube-apiserver /usr/local/bin/kube-apiserver
Step #2: #6 sha256:d625203475d3157e3aeb767c683de8d6c77f61db7153c37eb816150e06aa1ba8
Step #2: #5 transferring context: 142.41MB 1.5s done
Step #2: #5 DONE 1.7s
Step #2: 
Step #2: #4 [1/8] FROM docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350
Step #2: #4 sha256:c3005ada81c737c561a49ebcd38bc23df2db163a25aa637cb52527909f3d0e30
Step #2: #4 sha256:5820a41969ced8aea1e0fbb39cfc0dcc1422159e9f407e779151b9360c0a551f 1.46kB / 1.46kB done
Step #2: #4 sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350 1.85kB / 1.85kB done
Step #2: #4 sha256:137c6fa623edfe10559df66072d49958f2fe2becac23b851cb647fa37af693c2 529B / 529B done
Step #2: #4 sha256:dd4fab84af3401c34bae6e087ed1d66517b94a562b9875ac220e3e763bb3d5c4 8.39MB / 22.72MB 1.6s
Step #2: #4 sha256:dd4fab84af3401c34bae6e087ed1d66517b94a562b9875ac220e3e763bb3d5c4 22.72MB / 22.72MB 1.7s
Step #2: #4 extracting sha256:dd4fab84af3401c34bae6e087ed1d66517b94a562b9875ac220e3e763bb3d5c4
Step #2: #6 extracting sha256:920dd5dc200ba523d4d8395eab01fc977e4507a87765a66b59552224c6a4c22f 0.7s done
Step #2: #4 sha256:dd4fab84af3401c34bae6e087ed1d66517b94a562b9875ac220e3e763bb3d5c4 22.72MB / 22.72MB 1.9s done
Step #2: #6 extracting sha256:538547ed7102bd8629ea842d87f0345076c84bee08cf0fa41dad3f1ddaf5f4a8
Step #2: #6 extracting sha256:538547ed7102bd8629ea842d87f0345076c84bee08cf0fa41dad3f1ddaf5f4a8 0.3s done
Step #2: #6 extracting sha256:165d8763efcc37f35d190368bd2d4522e21584f8e9850f25d3de9ab04dbdd978
Step #2: #4 extracting sha256:dd4fab84af3401c34bae6e087ed1d66517b94a562b9875ac220e3e763bb3d5c4 1.1s done
Step #2: #6 extracting sha256:165d8763efcc37f35d190368bd2d4522e21584f8e9850f25d3de9ab04dbdd978 0.1s done
Step #2: #4 DONE 3.5s
Step #2: 
Step #2: #6 [2/8] COPY ginkgo /usr/local/bin/
Step #2: #6 sha256:fed7b8c457cd8fe5c5caf6596f56ba312f04f2719730e74a1b90828f362934ca
Step #2: #6 extracting sha256:0044f1d60d03253b06d22435ba89506a033d769d77b70d0547570ce37db2015a done
Step #2: #6 extracting sha256:c8381f480814de4b1bb2d189befa5a3318ad99fe19b92882680ee689b35b3e17 done
Step #2: #6 extracting sha256:7d49b9bee91c70bbb99661bd85d55fa30b7ade453a7c349eb7d0cffa60056160 done
Step #2: #6 DONE 6.9s
Step #2: 
Step #2: #4 [2/2] COPY kube-proxy /usr/local/bin/kube-proxy
Step #2: #4 sha256:9420e819ef33b4a30b3ca5771d946b07762ff1bd3900eee604d28e427bc35ecb
Step #2: #4 DONE 6.2s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #7 exporting layers 0.9s done
Step #2: #7 writing image sha256:1d4f593a7875c52b4ab9850950cbeaf7c5cdcfafcf832620f6a30a2c2dba0ab4 done
Step #2: #7 naming to k8s.gcr.io/kube-scheduler-arm:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.9s
Step #2: #6 DONE 6.4s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #4 DONE 1.0s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #6 DONE 6.1s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #6 DONE 4.0s
Step #2: 
Step #2: #7 [3/8] COPY e2e.test /usr/local/bin/
Step #2: #7 sha256:b568b61f868d974c65543e49b815647df1d60582fa715980033a402732c3f71f
Step #2: #7 exporting layers 0.5s done
Step #2: #7 DONE 0.7s
Step #2: #7 exporting layers 0.7s done
Step #2: #7 writing image sha256:415f7d986fd4830494ce1226297d169448faa955a1b92350df194ab788a0ea78 done
Step #2: #7 naming to k8s.gcr.io/kube-proxy-arm:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.7s
Step #2: #7 exporting layers 0.7s done
Step #2: #7 writing image sha256:25731cc75ace00aa3e429a9f92e849a263ecdb9cfe398afc98c65744f406327f done
Step #2: #7 naming to k8s.gcr.io/kube-controller-manager-arm:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.7s
Step #2: #7 writing image sha256:ff630c919194083d6256cc879d14f8c484752549e870e32f2d6953d6bf2c65a8 done
Step #2: #7 naming to k8s.gcr.io/kube-apiserver-arm:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.7s
Step #2: +++ [0129 14:32:56] Deleting docker image k8s.gcr.io/kube-scheduler-arm:v1.21.0-alpha.3
Step #2: 
Step #2: #8 [4/8] COPY kubectl /usr/local/bin/
Step #2: #8 sha256:f274e74f9108eb93fc76271efdd7c4c6bf176d8731c5daf29452d41d3a4881dd
Step #2: #8 DONE 1.3s
Step #2: 
Step #2: #9 [5/8] COPY run_e2e.sh /run_e2e.sh
Step #2: #9 sha256:6f46e80ce6545be1c1797861c2baaff998656e3d116a204cf5fe0975a2e98ae6
Step #2: #9 DONE 0.2s
Step #2: 
Step #2: #10 [6/8] COPY gorunner /gorunner
Step #2: #10 sha256:95dbada687b32f456003133c332cb7345c8113a86e7361ab7d37075071c79224
Step #2: +++ [0129 14:32:57] Deleting docker image k8s.gcr.io/kube-controller-manager-arm:v1.21.0-alpha.3
Step #2: +++ [0129 14:32:57] Deleting docker image k8s.gcr.io/kube-proxy-arm:v1.21.0-alpha.3
Step #2: +++ [0129 14:32:57] Deleting docker image k8s.gcr.io/kube-apiserver-arm:v1.21.0-alpha.3
Step #2: #10 DONE 1.2s
Step #2: 
Step #2: #11 [7/8] COPY cluster /kubernetes/cluster
Step #2: #11 sha256:7d74c2f67e11479997abfcbf62c4446b3565b5d917253b7ef75d1c707af6e909
Step #2: #11 DONE 0.1s
Step #2: 
Step #2: #12 [8/8] WORKDIR /usr/local/bin
Step #2: #12 sha256:67827ca91ee1ca38a437253423cb6c7073ae98df97c82b762be4f1bf1a28445e
Step #2: #12 DONE 0.0s
Step #2: 
Step #2: #13 exporting to image
Step #2: #13 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #13 exporting layers
Step #2: #13 exporting layers 0.6s done
Step #2: #13 writing image sha256:c4df32b94cf4e17cf46c393007cd362a45bf22e6c695706e7e309d549596a364 done
Step #2: #13 naming to k8s.gcr.io/conformance-arm:v1.21.0-alpha.3 done
Step #2: #13 DONE 0.6s
Step #2: +++ [0129 14:32:59] Building tarball: node linux-arm64
Step #2: +++ [0129 14:33:03] Deleting conformance image k8s.gcr.io/conformance-arm:v1.21.0-alpha.3
Step #2: +++ [0129 14:33:03] Docker builds done
Step #2: +++ [0129 14:33:03] Building images: linux-arm64
Step #2: +++ [0129 14:33:03] Starting docker build for image: kube-apiserver-arm64
Step #2: +++ [0129 14:33:03] Starting docker build for image: kube-controller-manager-arm64
Step #2: +++ [0129 14:33:03] Starting docker build for image: kube-scheduler-arm64
Step #2: +++ [0129 14:33:03] Starting docker build for image: kube-proxy-arm64
Step #2: +++ [0129 14:33:03] Building conformance image for arch: arm64
Step #2: time="2021-01-29T14:33:03Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:33:03Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:33:03Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:33:03Z" level=warning msg="quiet currently not implemented"
Step #2: #2 [internal] load .dockerignore
Step #2: #2 sha256:7600e7a6f156cbbe3fee65c51fd8b6827d320c857e3168e9767e3e64dcab25ba
Step #2: #2 transferring context: 2B done
Step #2: #2 DONE 0.1s
Step #2: 
Step #2: #1 [internal] load build definition from Dockerfile
Step #2: #1 sha256:8bc93c791ecdf6a98b932c4322b2fc046a35b49d571019cd74c87e87b57c7954
Step #2: #1 transferring dockerfile: 139B done
Step #2: #1 DONE 0.1s
Step #2: #2 [internal] load build definition from Dockerfile
Step #2: #2 sha256:c0e968daf1f1305fdf6f5be79fb6e87e9d29ad10e3d8edf7aa87f3fb0503b243
Step #2: #2 transferring dockerfile: 137B done
Step #2: #2 DONE 0.1s
Step #2: 
Step #2: #1 [internal] load .dockerignore
Step #2: #1 sha256:8ba11816185a96f570bdb186cb6abd47548f45d262960c43288152ad5bfb9c0d
Step #2: #1 transferring context: 2B done
Step #2: #1 DONE 0.1s
Step #2: #2 [internal] load build definition from Dockerfile
Step #2: #2 sha256:74e31291f2f511bfe9f2188a64af6d8a60cfe4e6f35a14a31ee44d9cf0219c21
Step #2: #2 transferring dockerfile: 157B done
Step #2: #2 DONE 0.1s
Step #2: 
Step #2: #1 [internal] load .dockerignore
Step #2: #1 sha256:7299e4edcc07edc3af59256bbbc08b0b520311f3cb7f4327f4a910a6cd971292
Step #2: #1 transferring context: 2B done
Step #2: #1 DONE 0.1s
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:dde651d2a4b0439803f83d6fbc5173e3395f4abcb7d1e39e42ae77d4b62c9a1d
Step #2: #2 [internal] load build definition from Dockerfile
Step #2: #2 sha256:c035cacc9ecb285458d84f81c5116c6caf4de2327f4e70a0ed48b788647d5763
Step #2: #2 transferring dockerfile: 139B done
Step #2: #2 DONE 0.0s
Step #2: 
Step #2: #1 [internal] load .dockerignore
Step #2: #1 sha256:b7697b4b6a27b47b3320b40b4c0c9e81054075a96b88bd5f42703921e609304e
Step #2: #1 transferring context: 2B done
Step #2: #1 DONE 0.1s
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:dde651d2a4b0439803f83d6fbc5173e3395f4abcb7d1e39e42ae77d4b62c9a1d
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:dde651d2a4b0439803f83d6fbc5173e3395f4abcb7d1e39e42ae77d4b62c9a1d
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0
Step #2: #3 sha256:dfaadc7bb474f00e59dcb592dadab793d5637fa496b387da32f0597aa95b35a5
Step #2: #2 [internal] load .dockerignore
Step #2: #2 sha256:e1d4eb2c333528d3586bf226d6ce7fcd8243b74fff6b174bc9c4738524ee99f7
Step #2: #2 DONE 0.0s
Step #2: 
Step #2: #1 [internal] load build definition from Dockerfile
Step #2: #1 sha256:189325750f55e2ce9cc9b7823b8d74cfabf1c8097707bee8db6568560a878f17
Step #2: #3 DONE 0.5s
Step #2: #1 transferring dockerfile: 1.05kB done
Step #2: #1 DONE 0.0s
Step #2: 
Step #2: #2 [internal] load .dockerignore
Step #2: #2 sha256:e1d4eb2c333528d3586bf226d6ce7fcd8243b74fff6b174bc9c4738524ee99f7
Step #2: #2 transferring context: 2B done
Step #2: #2 DONE 0.0s
Step #2: 
Step #2: #3 [internal] load metadata for docker.io/library/debian:stable-slim
Step #2: #3 sha256:18e160d71c89b5192a95ab017c198f9acbdacb3608f3d4df592e7eb9385b4ca3
Step #2: #3 DONE 0.5s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb
Step #2: #4 sha256:d449de657fbdd66c96349d01789e91634cb35ff45ef883f77d55568cf2931c2f
Step #2: #4 resolve k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb done
Step #2: #3 DONE 0.5s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:0add35f6dbf859a7426d6ae5e73475c05b9b4dc8ee371a2a3d1d848e18c3ef21
Step #2: #4 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:0add35f6dbf859a7426d6ae5e73475c05b9b4dc8ee371a2a3d1d848e18c3ef21
Step #2: #4 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: #3 DONE 0.5s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:0add35f6dbf859a7426d6ae5e73475c05b9b4dc8ee371a2a3d1d848e18c3ef21
Step #2: #4 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: #4 sha256:1688aa37b99a7aaad266c52d2856c35e79cb5872239e3faca402cdfce0d4e439 0B / 649.10kB 0.1s
Step #2: #4 sha256:6492ff17dda94dda0c6861922790fab69a8d78c5b3cebb33b63d540570a57031 0B / 669.62kB 0.1s
Step #2: #4 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #4 sha256:4b8dcdf86964ae8ceea7eca409ca7b87ad01421c1db0a0a99cc99676785496e3 737B / 737B done
Step #2: #4 sha256:8c801e4e8e8e86477c927ee5fa1dbec69b830de328468d37345719e37ff0b571 1.42kB / 1.42kB done
Step #2: #4 sha256:1688aa37b99a7aaad266c52d2856c35e79cb5872239e3faca402cdfce0d4e439 0B / 649.10kB 0.1s
Step #2: #4 sha256:6492ff17dda94dda0c6861922790fab69a8d78c5b3cebb33b63d540570a57031 0B / 669.62kB 0.1s
Step #2: #4 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #4 sha256:4b8dcdf86964ae8ceea7eca409ca7b87ad01421c1db0a0a99cc99676785496e3 737B / 737B done
Step #2: #4 sha256:8c801e4e8e8e86477c927ee5fa1dbec69b830de328468d37345719e37ff0b571 1.42kB / 1.42kB done
Step #2: #4 sha256:1688aa37b99a7aaad266c52d2856c35e79cb5872239e3faca402cdfce0d4e439 0B / 649.10kB 0.1s
Step #2: #4 sha256:6492ff17dda94dda0c6861922790fab69a8d78c5b3cebb33b63d540570a57031 0B / 669.62kB 0.1s
Step #2: #4 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #4 sha256:4b8dcdf86964ae8ceea7eca409ca7b87ad01421c1db0a0a99cc99676785496e3 737B / 737B done
Step #2: #4 sha256:8c801e4e8e8e86477c927ee5fa1dbec69b830de328468d37345719e37ff0b571 1.42kB / 1.42kB done
Step #2: #4 sha256:b4fa11965f34a9f668c424b401c0af22e88f600d22c899699bdb0bd1e6953ad6 1.57kB / 1.57kB done
Step #2: #4 sha256:0d23a9363a675efc4be1e914c180f7aec2b6a51ec0476b23e5313d59d7f06d12 2.68kB / 2.68kB done
Step #2: #4 sha256:6022c6b01d76e534f6a2b2c0db0de9f3585801c1d78437daaceee6fcce5a1325 0B / 23.57MB 0.1s
Step #2: #4 sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb 1.70kB / 1.70kB done
Step #2: #4 sha256:1688aa37b99a7aaad266c52d2856c35e79cb5872239e3faca402cdfce0d4e439 649.10kB / 649.10kB 0.3s done
Step #2: #4 sha256:6492ff17dda94dda0c6861922790fab69a8d78c5b3cebb33b63d540570a57031 669.62kB / 669.62kB 0.3s done
Step #2: #4 extracting sha256:1688aa37b99a7aaad266c52d2856c35e79cb5872239e3faca402cdfce0d4e439
Step #2: #4 sha256:1688aa37b99a7aaad266c52d2856c35e79cb5872239e3faca402cdfce0d4e439 649.10kB / 649.10kB 0.3s done
Step #2: #4 sha256:6492ff17dda94dda0c6861922790fab69a8d78c5b3cebb33b63d540570a57031 669.62kB / 669.62kB 0.3s done
Step #2: #4 extracting sha256:1688aa37b99a7aaad266c52d2856c35e79cb5872239e3faca402cdfce0d4e439
Step #2: #3 DONE 0.5s
Step #2: #4 sha256:1688aa37b99a7aaad266c52d2856c35e79cb5872239e3faca402cdfce0d4e439 649.10kB / 649.10kB 0.3s done
Step #2: #4 sha256:6492ff17dda94dda0c6861922790fab69a8d78c5b3cebb33b63d540570a57031 669.62kB / 669.62kB 0.3s done
Step #2: #4 extracting sha256:1688aa37b99a7aaad266c52d2856c35e79cb5872239e3faca402cdfce0d4e439
Step #2: #4 ...
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:3466041f7346029766a0c3c5e41f48815cda3da558044ff9c5bfaf3dc6e4edd4
Step #2: #5 transferring context: 36.64MB 0.5s done
Step #2: #5 DONE 0.5s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb
Step #2: #4 sha256:d449de657fbdd66c96349d01789e91634cb35ff45ef883f77d55568cf2931c2f
Step #2: #4 sha256:34ec07668ea1c5997cce3a5991404074e0d34fd2c2b4e4a43e6fd206cd010777 0B / 14.71MB 0.4s
Step #2: #4 sha256:87143f7506d04b66c3290564d2f35b39da63cc85c8c408fb00fa95a1b384a954 0B / 1.84MB 0.4s
Step #2: #4 ...
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:e1152b90a0347418899f603947e83f68c6790a11acd1bbc1113aa48525721938
Step #2: #5 transferring context: 39.79MB 0.4s done
Step #2: #5 DONE 0.5s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:0add35f6dbf859a7426d6ae5e73475c05b9b4dc8ee371a2a3d1d848e18c3ef21
Step #2: 
Step #2: #4 [1/8] FROM docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350
Step #2: #4 sha256:2a0d5d1913052b038279b61ef94f6afaa691f32c4051dfff51efa1ad844d1038
Step #2: #4 resolve docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350 done
Step #2: #4 sha256:6022c6b01d76e534f6a2b2c0db0de9f3585801c1d78437daaceee6fcce5a1325 8.39MB / 23.57MB 0.6s
Step #2: #4 sha256:87143f7506d04b66c3290564d2f35b39da63cc85c8c408fb00fa95a1b384a954 1.84MB / 1.84MB 0.6s
Step #2: #4 extracting sha256:1688aa37b99a7aaad266c52d2856c35e79cb5872239e3faca402cdfce0d4e439 0.3s done
Step #2: #4 extracting sha256:1688aa37b99a7aaad266c52d2856c35e79cb5872239e3faca402cdfce0d4e439 0.3s done
Step #2: #4 sha256:212e58ee7e78661b721c4720aaa7fd8ebdc80d159461e53fd6b67db178fc1606 1.46kB / 1.46kB done
Step #2: #4 sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350 1.85kB / 1.85kB done
Step #2: #4 sha256:04cb6d16fd0c6c495bf7972bf659acddb8be4741e2731ac538fea34568021928 529B / 529B done
Step #2: #4 extracting sha256:1688aa37b99a7aaad266c52d2856c35e79cb5872239e3faca402cdfce0d4e439 0.3s done
Step #2: #4 extracting sha256:6492ff17dda94dda0c6861922790fab69a8d78c5b3cebb33b63d540570a57031
Step #2: #4 sha256:87143f7506d04b66c3290564d2f35b39da63cc85c8c408fb00fa95a1b384a954 1.84MB / 1.84MB 0.6s done
Step #2: #4 sha256:06779b4c74236f8eea30d4fb06b65f8001428c74c8490a987d10211a866df539 0B / 1.03kB 0.7s
Step #2: #4 extracting sha256:6492ff17dda94dda0c6861922790fab69a8d78c5b3cebb33b63d540570a57031
Step #2: #4 extracting sha256:6492ff17dda94dda0c6861922790fab69a8d78c5b3cebb33b63d540570a57031 0.1s done
Step #2: #4 extracting sha256:6492ff17dda94dda0c6861922790fab69a8d78c5b3cebb33b63d540570a57031 0.1s done
Step #2: #4 sha256:6022c6b01d76e534f6a2b2c0db0de9f3585801c1d78437daaceee6fcce5a1325 23.57MB / 23.57MB 0.9s
Step #2: #4 sha256:34ec07668ea1c5997cce3a5991404074e0d34fd2c2b4e4a43e6fd206cd010777 8.39MB / 14.71MB 0.9s
Step #2: #4 sha256:06779b4c74236f8eea30d4fb06b65f8001428c74c8490a987d10211a866df539 1.03kB / 1.03kB 0.8s done
Step #2: #4 sha256:a6785c7b362900613462a8e74fe2b8feef4377e5da599e24efae24f2669ce9e4 0B / 519B 0.9s
Step #2: #4 extracting sha256:6492ff17dda94dda0c6861922790fab69a8d78c5b3cebb33b63d540570a57031 0.1s done
Step #2: #4 DONE 0.9s
Step #2: 
Step #2: #6 [2/2] COPY kube-scheduler /usr/local/bin/kube-scheduler
Step #2: #6 sha256:558655a1855d248c302e279100b43960fb65767857ab28034723ee29cb625d05
Step #2: #4 DONE 0.9s
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:32c9394244eec0cf08ff012ad148d298af897069fed85b59bbcdb21cae621799
Step #2: #4 DONE 0.9s
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:305e2218b7ecbfc01398523f70dbe73640b1378a629e705ae9713ebebd03f2a7
Step #2: #4 extracting sha256:6022c6b01d76e534f6a2b2c0db0de9f3585801c1d78437daaceee6fcce5a1325
Step #2: #5 transferring context: 109.53MB 1.3s done
Step #2: #5 transferring context: 104.42MB 1.2s done
Step #2: #4 sha256:1e2e99bba8f62d0f8c5565a724281fb3a33754b47dfa7d279d5c7eab2bb12517 0B / 25.86MB 1.1s
Step #2: #4 sha256:6022c6b01d76e534f6a2b2c0db0de9f3585801c1d78437daaceee6fcce5a1325 23.57MB / 23.57MB 0.9s done
Step #2: #4 sha256:34ec07668ea1c5997cce3a5991404074e0d34fd2c2b4e4a43e6fd206cd010777 14.71MB / 14.71MB 1.2s done
Step #2: #4 sha256:a6785c7b362900613462a8e74fe2b8feef4377e5da599e24efae24f2669ce9e4 519B / 519B 1.3s done
Step #2: #4 sha256:2113453aa577fdee885360bbbc552fe8511740578b1cde262cb815afd4be8f68 0B / 548B 1.6s
Step #2: #4 sha256:2113453aa577fdee885360bbbc552fe8511740578b1cde262cb815afd4be8f68 548B / 548B 1.7s
Step #2: #4 sha256:2113453aa577fdee885360bbbc552fe8511740578b1cde262cb815afd4be8f68 548B / 548B 1.8s done
Step #2: #5 DONE 2.0s
Step #2: #4 sha256:1e2e99bba8f62d0f8c5565a724281fb3a33754b47dfa7d279d5c7eab2bb12517 8.39MB / 25.86MB 1.5s
Step #2: #4 extracting sha256:6022c6b01d76e534f6a2b2c0db0de9f3585801c1d78437daaceee6fcce5a1325 0.9s done
Step #2: #4 sha256:1e2e99bba8f62d0f8c5565a724281fb3a33754b47dfa7d279d5c7eab2bb12517 23.48MB / 25.86MB 1.8s
Step #2: #4 sha256:1e2e99bba8f62d0f8c5565a724281fb3a33754b47dfa7d279d5c7eab2bb12517 25.86MB / 25.86MB 1.9s
Step #2: #4 extracting sha256:1e2e99bba8f62d0f8c5565a724281fb3a33754b47dfa7d279d5c7eab2bb12517
Step #2: 
Step #2: #6 [2/2] COPY kube-controller-manager /usr/local/bin/kube-controller-manager
Step #2: #6 sha256:7e219649046896d2429c8390184e1cd6c8600f7ace2d3a39484f3b1720b393fc
Step #2: #4 sha256:1e2e99bba8f62d0f8c5565a724281fb3a33754b47dfa7d279d5c7eab2bb12517 25.86MB / 25.86MB 2.1s done
Step #2: #4 extracting sha256:34ec07668ea1c5997cce3a5991404074e0d34fd2c2b4e4a43e6fd206cd010777
Step #2: #4 ...
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:c118f32e5f1fb577ad858312ae68dc3785ee6685a6f0c43232ecb06cb54469ab
Step #2: #5 transferring context: 160.95MB 1.5s done
Step #2: #5 DONE 2.4s
Step #2: 
Step #2: #4 [1/8] FROM docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350
Step #2: #4 sha256:2a0d5d1913052b038279b61ef94f6afaa691f32c4051dfff51efa1ad844d1038
Step #2: #5 DONE 2.8s
Step #2: 
Step #2: #6 [2/2] COPY kube-apiserver /usr/local/bin/kube-apiserver
Step #2: #6 sha256:6da0a2c4a900817d869bf98015acafed47d5ee138cf6f18af7d231b98f639350
Step #2: #4 extracting sha256:34ec07668ea1c5997cce3a5991404074e0d34fd2c2b4e4a43e6fd206cd010777 0.3s done
Step #2: #4 extracting sha256:87143f7506d04b66c3290564d2f35b39da63cc85c8c408fb00fa95a1b384a954
Step #2: #4 extracting sha256:87143f7506d04b66c3290564d2f35b39da63cc85c8c408fb00fa95a1b384a954 0.1s done
Step #2: #4 extracting sha256:1e2e99bba8f62d0f8c5565a724281fb3a33754b47dfa7d279d5c7eab2bb12517 1.1s done
Step #2: #4 extracting sha256:06779b4c74236f8eea30d4fb06b65f8001428c74c8490a987d10211a866df539 done
Step #2: #4 DONE 3.6s
Step #2: #4 extracting sha256:a6785c7b362900613462a8e74fe2b8feef4377e5da599e24efae24f2669ce9e4
Step #2: 
Step #2: #6 [2/8] COPY ginkgo /usr/local/bin/
Step #2: #6 sha256:9f7db3f1bbc66fbdb769097692c6123a0c50fd6bc42287cfa79a51537bf1f70a
Step #2: #4 extracting sha256:a6785c7b362900613462a8e74fe2b8feef4377e5da599e24efae24f2669ce9e4 done
Step #2: #4 extracting sha256:2113453aa577fdee885360bbbc552fe8511740578b1cde262cb815afd4be8f68
Step #2: #4 extracting sha256:2113453aa577fdee885360bbbc552fe8511740578b1cde262cb815afd4be8f68 done
Step #2: #4 DONE 6.4s
Step #2: 
Step #2: #6 [2/2] COPY kube-proxy /usr/local/bin/kube-proxy
Step #2: #6 sha256:99a8d677a4ba325d523f220c0859e7482e11a56c75553b3a32f3d161c994653f
Step #2: #6 DONE 5.7s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #7 exporting layers 0.8s done
Step #2: #7 writing image sha256:b56c2dafe531418da4a083d2422c0b8ed4af41f13b3f57f0a3a8dbd731920175 0.0s done
Step #2: #7 naming to k8s.gcr.io/kube-scheduler-arm64:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.9s
Step #2: #6 DONE 3.5s
Step #2: #6 DONE 4.9s
Step #2: #6 DONE 4.8s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #6 DONE 1.2s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: 
Step #2: #7 [3/8] COPY e2e.test /usr/local/bin/
Step #2: #7 sha256:ea8b7cce87f4a91349d42b898b5180dce31accedb4c3c7a714bde34d0ef837aa
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #7 exporting layers 0.7s done
Step #2: #7 writing image sha256:13d4d075cad6d9bf57f50dde8686bd606b1ef24f4d76e819ce79eec4237dfc32 done
Step #2: #7 naming to k8s.gcr.io/kube-proxy-arm64:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.8s
Step #2: #7 exporting layers 0.7s done
Step #2: #7 writing image sha256:d293a707d1c2e65ac63e8411f35a7cd91ecd3f20c6536b1c4d84c2460e2ec424 done
Step #2: #7 naming to k8s.gcr.io/kube-controller-manager-arm64:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.8s
Step #2: #7 exporting layers 0.7s done
Step #2: #7 writing image sha256:4fdae9aa7933b879e64dd074f2d0220381e46badf8634cedeb74cb2c4fe378d6
Step #2: #7 writing image sha256:4fdae9aa7933b879e64dd074f2d0220381e46badf8634cedeb74cb2c4fe378d6 done
Step #2: #7 naming to k8s.gcr.io/kube-apiserver-arm64:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.8s
Step #2: +++ [0129 14:33:12] Deleting docker image k8s.gcr.io/kube-scheduler-arm64:v1.21.0-alpha.3
Step #2: #7 DONE 0.8s
Step #2: 
Step #2: #8 [4/8] COPY kubectl /usr/local/bin/
Step #2: #8 sha256:365e90932b33728584939b11a1efe71b4fdadaf5578a45ab9703846ccd4e87a3
Step #2: +++ [0129 14:33:14] Building tarball: node linux-s390x
Step #2: #8 DONE 1.6s
Step #2: 
Step #2: #9 [5/8] COPY run_e2e.sh /run_e2e.sh
Step #2: #9 sha256:fdc8fe2cac5fd92b13b1a18b6effec0aa8fcc681e79fb5c213ff4fdbd73db17a
Step #2: #9 DONE 1.4s
Step #2: 
Step #2: #10 [6/8] COPY gorunner /gorunner
Step #2: #10 sha256:a5758cd11c2a5c846b46a78cc022725e13e99de83244b7cb21a9a98615689452
Step #2: +++ [0129 14:33:15] Deleting docker image k8s.gcr.io/kube-controller-manager-arm64:v1.21.0-alpha.3
Step #2: +++ [0129 14:33:15] Deleting docker image k8s.gcr.io/kube-apiserver-arm64:v1.21.0-alpha.3
Step #2: +++ [0129 14:33:15] Deleting docker image k8s.gcr.io/kube-proxy-arm64:v1.21.0-alpha.3
Step #2: #10 DONE 1.4s
Step #2: 
Step #2: #11 [7/8] COPY cluster /kubernetes/cluster
Step #2: #11 sha256:25f22b3e281859f0e3bb0a72d031fba2da48b51e77e58c636e0723137a183785
Step #2: #11 DONE 0.1s
Step #2: 
Step #2: #12 [8/8] WORKDIR /usr/local/bin
Step #2: #12 sha256:234e631c11eb4ea0ad2e58df7846d5c2a026da5edc66b8d73e0487b2a7a9c842
Step #2: #12 DONE 0.0s
Step #2: 
Step #2: #13 exporting to image
Step #2: #13 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #13 exporting layers
Step #2: #13 exporting layers 0.7s done
Step #2: #13 writing image sha256:3fda1e8a84ce5ac82483a9f29c441c9773ba2f7214021fca5a319c4c3222f43c done
Step #2: #13 naming to k8s.gcr.io/conformance-arm64:v1.21.0-alpha.3 done
Step #2: #13 DONE 0.7s
Step #2: +++ [0129 14:33:20] Deleting conformance image k8s.gcr.io/conformance-arm64:v1.21.0-alpha.3
Step #2: +++ [0129 14:33:20] Docker builds done
Step #2: +++ [0129 14:33:20] Building images: linux-s390x
Step #2: +++ [0129 14:33:20] Starting docker build for image: kube-apiserver-s390x
Step #2: +++ [0129 14:33:20] Starting docker build for image: kube-controller-manager-s390x
Step #2: +++ [0129 14:33:20] Starting docker build for image: kube-scheduler-s390x
Step #2: +++ [0129 14:33:20] Starting docker build for image: kube-proxy-s390x
Step #2: +++ [0129 14:33:20] Building conformance image for arch: s390x
Step #2: time="2021-01-29T14:33:21Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:33:21Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:33:21Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:33:21Z" level=warning msg="quiet currently not implemented"
Step #2: #2 [internal] load build definition from Dockerfile
Step #2: #2 sha256:84dfedda24ae5f8dfd92c4a40f6a26a57066eed4fe606629ed971f2c59f34ff3
Step #2: #2 [internal] load build definition from Dockerfile
Step #2: #2 sha256:bd614cd7be0a7df371c4d740b25cd6b6e1fd3a832f1321cb64e292685950c8e8
Step #2: #2 transferring dockerfile: 139B done
Step #2: #2 DONE 0.1s
Step #2: 
Step #2: #1 [internal] load .dockerignore
Step #2: #1 sha256:091c9ccec03cff29b12eae7662a07d3ba741e46d8dd3a7af76923e2850804801
Step #2: #1 transferring context: 2B done
Step #2: #1 DONE 0.1s
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:8e3ca16f4b93495a5adedfe82c27b9e783df903a85cecad85fd8ce3b0cd10b9c
Step #2: #2 transferring dockerfile: 157B done
Step #2: #2 DONE 0.1s
Step #2: 
Step #2: #1 [internal] load .dockerignore
Step #2: #1 sha256:616f3c798cc38a1c5f023db4c4978f2276dd0a1db4c599bb0137044e2600b37e
Step #2: #1 transferring context: 2B done
Step #2: #1 DONE 0.1s
Step #2: #2 [internal] load .dockerignore
Step #2: #2 sha256:b650cb0b25b3f47ad5a4f6161aba7bf96f217c6675eb5a95103939e7c57b2208
Step #2: #2 transferring context: 2B done
Step #2: #2 DONE 0.0s
Step #2: 
Step #2: #1 [internal] load build definition from Dockerfile
Step #2: #1 sha256:1d3eb4d7e970a0fa70447c2bc2171931d006abeaaf33b7c9f0eea1d54d0da553
Step #2: #1 transferring dockerfile: 137B done
Step #2: #1 DONE 0.0s
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0
Step #2: #3 sha256:14901df6c6a9c551bbf9b192a61e297c4be89df2525f3d0f04e91b8d55099e0a
Step #2: #1 [internal] load .dockerignore
Step #2: #1 sha256:f49d3b7f4f184dc8b0cebbe5ccd82b73769dfcac63e5c7ac855e276a30a4c130
Step #2: #1 transferring context: 2B done
Step #2: #1 DONE 0.1s
Step #2: 
Step #2: #2 [internal] load build definition from Dockerfile
Step #2: #2 sha256:1441134f02279e66d95266b739038a85b531ee7006af67fdedb099aa674ed8a2
Step #2: #2 transferring dockerfile: 139B done
Step #2: #2 DONE 0.1s
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:8e3ca16f4b93495a5adedfe82c27b9e783df903a85cecad85fd8ce3b0cd10b9c
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:8e3ca16f4b93495a5adedfe82c27b9e783df903a85cecad85fd8ce3b0cd10b9c
Step #2: #3 DONE 0.4s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:63ac6c051c62d980affb1fa79cb006df55b7b8a292dadeebc08166cf966c2d5e
Step #2: #4 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: #2 [internal] load .dockerignore
Step #2: #2 sha256:df6388cc4e0fb3202ae2ba6531d9c9e0db83b436fe4c6eed8bb35ea250901b1e
Step #2: #2 DONE 0.0s
Step #2: #3 DONE 0.4s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:63ac6c051c62d980affb1fa79cb006df55b7b8a292dadeebc08166cf966c2d5e
Step #2: #4 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: #3 DONE 0.4s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb
Step #2: #4 sha256:6054eef2207852ca638cf534d88e1a67dbfadef19116db202da8f1214a4a2005
Step #2: #4 resolve k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb done
Step #2: #3 DONE 0.4s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:63ac6c051c62d980affb1fa79cb006df55b7b8a292dadeebc08166cf966c2d5e
Step #2: #4 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: 
Step #2: #2 [internal] load .dockerignore
Step #2: #2 sha256:df6388cc4e0fb3202ae2ba6531d9c9e0db83b436fe4c6eed8bb35ea250901b1e
Step #2: #2 transferring context: 2B done
Step #2: #2 DONE 0.0s
Step #2: 
Step #2: #1 [internal] load build definition from Dockerfile
Step #2: #1 sha256:103dbb156ef2d3580832f7a575b09df7a8bcaa0c28b7154cc18aa7e2fe82fa3c
Step #2: #1 transferring dockerfile: 1.05kB done
Step #2: #1 DONE 0.0s
Step #2: 
Step #2: #3 [internal] load metadata for docker.io/library/debian:stable-slim
Step #2: #3 sha256:05c6b11bb62141f367bb5bdc208053faedce33b3b92e05273321a1156262ef8d
Step #2: #4 sha256:50ef25fba428b6002ef0a9dea7ceae5045430dc1035d50498a478eefccba17f5 1.57kB / 1.57kB done
Step #2: #4 sha256:b859056ca3ea66ecd0618de06898a51c19d7cd1086824a4b4b0579865270e852 2.69kB / 2.69kB done
Step #2: #4 sha256:376d61ddc98ab78b241628e7663b118ecccbc700046029c73e2a333219d21bce 0B / 23.06MB 0.1s
Step #2: #4 sha256:0ea90407440f407bf09809314aa15cdae15c80cf69ae730af49d75384c35e425 0B / 14.44MB 0.1s
Step #2: #4 sha256:bd8432a91c71af90df65ae153260f6fed26d7cd8fd2204da40d9ee140bfe081b 0B / 1.65MB 0.1s
Step #2: #4 sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb 1.70kB / 1.70kB done
Step #2: #4 sha256:9d72531013b119ac425a7790dc399f741e1fbcf98876fff8244996fc3cb394da 1.42kB / 1.42kB done
Step #2: #4 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #4 sha256:9ed1b51936ff6057d1534f5d22309d4e7369211d302b3bd72554e9d3963dbae3 737B / 737B done
Step #2: #4 sha256:9d72531013b119ac425a7790dc399f741e1fbcf98876fff8244996fc3cb394da 1.42kB / 1.42kB done
Step #2: #4 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #4 sha256:9ed1b51936ff6057d1534f5d22309d4e7369211d302b3bd72554e9d3963dbae3 737B / 737B done
Step #2: #4 sha256:9d72531013b119ac425a7790dc399f741e1fbcf98876fff8244996fc3cb394da 1.42kB / 1.42kB done
Step #2: #4 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #4 sha256:9ed1b51936ff6057d1534f5d22309d4e7369211d302b3bd72554e9d3963dbae3 737B / 737B done
Step #2: #4 ...
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:c545bca81cfa9db7ae09c651cfb623a2a77d7ff5d7d90d4a6d9516f52de62437
Step #2: #5 transferring context: 38.35MB 0.4s done
Step #2: #5 DONE 0.4s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb
Step #2: #4 sha256:6054eef2207852ca638cf534d88e1a67dbfadef19116db202da8f1214a4a2005
Step #2: #4 sha256:bd8432a91c71af90df65ae153260f6fed26d7cd8fd2204da40d9ee140bfe081b 1.65MB / 1.65MB 0.3s done
Step #2: #4 sha256:324bff1ba43c9fd3245497b39a42ff88ab08eacc61719c4c21c5af0626175f29 0B / 1.03kB 0.4s
Step #2: #4 ...
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:dc63475030071597d48064d935186d082eca76cdfba866162026f12aa72a922f
Step #2: #5 transferring context: 41.69MB 0.4s done
Step #2: #5 DONE 0.4s
Step #2: #3 DONE 0.5s
Step #2: 
Step #2: #4 [1/8] FROM docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350
Step #2: #4 sha256:b8a6234994ebf3acbd599cfb04d540a2e95b011f85bab1d2d18bfde7458e5951
Step #2: #4 resolve docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350 done
Step #2: #4 sha256:376d61ddc98ab78b241628e7663b118ecccbc700046029c73e2a333219d21bce 8.39MB / 23.06MB 0.6s
Step #2: #4 sha256:0ea90407440f407bf09809314aa15cdae15c80cf69ae730af49d75384c35e425 8.39MB / 14.44MB 0.6s
Step #2: #4 sha256:324bff1ba43c9fd3245497b39a42ff88ab08eacc61719c4c21c5af0626175f29 1.03kB / 1.03kB 0.5s done
Step #2: #4 sha256:7474d8eb55e498e9561206e7a8c10e7412d6692341daa29e969e40f6b507291d 0B / 521B 0.6s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:63ac6c051c62d980affb1fa79cb006df55b7b8a292dadeebc08166cf966c2d5e
Step #2: #4 sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350 1.85kB / 1.85kB done
Step #2: #4 sha256:9f886ce75f5ac4114b38a6b199a5cd359c00b1604fae97a1120d1c23844da4eb 529B / 529B done
Step #2: #4 sha256:d7d03459dc571308fb829cb6fe5c2383a59fbc332297886f78347351ee25a1d2 1.46kB / 1.46kB done
Step #2: #4 sha256:7474d8eb55e498e9561206e7a8c10e7412d6692341daa29e969e40f6b507291d 521B / 521B 0.7s done
Step #2: #4 sha256:ce1ed7d14c9093af116b2d5177d7f1553fb01ad59504b8050424614254274030 0B / 550B 0.8s
Step #2: #4 sha256:376d61ddc98ab78b241628e7663b118ecccbc700046029c73e2a333219d21bce 23.06MB / 23.06MB 1.0s
Step #2: #4 sha256:0ea90407440f407bf09809314aa15cdae15c80cf69ae730af49d75384c35e425 14.44MB / 14.44MB 0.9s done
Step #2: #4 sha256:ce1ed7d14c9093af116b2d5177d7f1553fb01ad59504b8050424614254274030 550B / 550B 0.9s done
Step #2: #4 ...
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:d7cf8f5159fbbe6f84772d853340959f4abb1f5e9c572675edd5cb2699586bd2
Step #2: #5 transferring context: 108.81MB 1.0s done
Step #2: #5 DONE 1.0s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:63ac6c051c62d980affb1fa79cb006df55b7b8a292dadeebc08166cf966c2d5e
Step #2: #4 sha256:0893082468c44575c6a05ffa0ee6a6e0b376c3b7ee0e9c7f20acdaf04edd67e5 0B / 649.10kB 1.0s
Step #2: #4 sha256:35cf5b5de6fb9f7c25081f8afbedf44ce20dd4b490d06cb105c38ed4e2c29eb8 0B / 699.20kB 1.0s
Step #2: #4 sha256:0893082468c44575c6a05ffa0ee6a6e0b376c3b7ee0e9c7f20acdaf04edd67e5 0B / 649.10kB 1.0s
Step #2: #4 sha256:35cf5b5de6fb9f7c25081f8afbedf44ce20dd4b490d06cb105c38ed4e2c29eb8 0B / 699.20kB 1.0s
Step #2: #4 sha256:a3590260a509ebfb98c639d22f55faa45de1287cdb2516fe5bba997d3c9fb578 0B / 25.72MB 0.5s
Step #2: #4 sha256:0893082468c44575c6a05ffa0ee6a6e0b376c3b7ee0e9c7f20acdaf04edd67e5 0B / 649.10kB 1.0s
Step #2: #4 sha256:35cf5b5de6fb9f7c25081f8afbedf44ce20dd4b490d06cb105c38ed4e2c29eb8 0B / 699.20kB 1.0s
Step #2: #4 sha256:376d61ddc98ab78b241628e7663b118ecccbc700046029c73e2a333219d21bce 23.06MB / 23.06MB 1.0s done
Step #2: #4 extracting sha256:376d61ddc98ab78b241628e7663b118ecccbc700046029c73e2a333219d21bce
Step #2: #4 ...
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:cd3f3590f5a505d89871f40a42a33a15d4edacddebab97251134e442559ca43b
Step #2: #5 transferring context: 113.73MB 1.1s done
Step #2: #5 DONE 1.1s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:63ac6c051c62d980affb1fa79cb006df55b7b8a292dadeebc08166cf966c2d5e
Step #2: #4 extracting sha256:0893082468c44575c6a05ffa0ee6a6e0b376c3b7ee0e9c7f20acdaf04edd67e5
Step #2: #4 sha256:0893082468c44575c6a05ffa0ee6a6e0b376c3b7ee0e9c7f20acdaf04edd67e5 649.10kB / 649.10kB 1.1s done
Step #2: #4 sha256:35cf5b5de6fb9f7c25081f8afbedf44ce20dd4b490d06cb105c38ed4e2c29eb8 699.20kB / 699.20kB 1.1s done
Step #2: #4 extracting sha256:0893082468c44575c6a05ffa0ee6a6e0b376c3b7ee0e9c7f20acdaf04edd67e5
Step #2: #4 sha256:0893082468c44575c6a05ffa0ee6a6e0b376c3b7ee0e9c7f20acdaf04edd67e5 649.10kB / 649.10kB 1.1s done
Step #2: #4 sha256:35cf5b5de6fb9f7c25081f8afbedf44ce20dd4b490d06cb105c38ed4e2c29eb8 699.20kB / 699.20kB 1.1s done
Step #2: #4 extracting sha256:0893082468c44575c6a05ffa0ee6a6e0b376c3b7ee0e9c7f20acdaf04edd67e5
Step #2: #4 sha256:0893082468c44575c6a05ffa0ee6a6e0b376c3b7ee0e9c7f20acdaf04edd67e5 649.10kB / 649.10kB 1.1s done
Step #2: #4 sha256:35cf5b5de6fb9f7c25081f8afbedf44ce20dd4b490d06cb105c38ed4e2c29eb8 699.20kB / 699.20kB 1.1s done
Step #2: #4 extracting sha256:0893082468c44575c6a05ffa0ee6a6e0b376c3b7ee0e9c7f20acdaf04edd67e5 0.1s done
Step #2: #4 sha256:a3590260a509ebfb98c639d22f55faa45de1287cdb2516fe5bba997d3c9fb578 2.06MB / 25.72MB 0.8s
Step #2: #4 extracting sha256:0893082468c44575c6a05ffa0ee6a6e0b376c3b7ee0e9c7f20acdaf04edd67e5 0.1s done
Step #2: #4 extracting sha256:0893082468c44575c6a05ffa0ee6a6e0b376c3b7ee0e9c7f20acdaf04edd67e5 0.1s done
Step #2: #4 sha256:a3590260a509ebfb98c639d22f55faa45de1287cdb2516fe5bba997d3c9fb578 8.39MB / 25.72MB 0.9s
Step #2: #4 extracting sha256:35cf5b5de6fb9f7c25081f8afbedf44ce20dd4b490d06cb105c38ed4e2c29eb8 0.1s done
Step #2: #4 extracting sha256:35cf5b5de6fb9f7c25081f8afbedf44ce20dd4b490d06cb105c38ed4e2c29eb8 0.1s done
Step #2: #4 DONE 1.6s
Step #2: #4 extracting sha256:35cf5b5de6fb9f7c25081f8afbedf44ce20dd4b490d06cb105c38ed4e2c29eb8 0.1s done
Step #2: #4 DONE 1.6s
Step #2: #4 DONE 1.6s
Step #2: 
Step #2: #6 [2/2] COPY kube-scheduler /usr/local/bin/kube-scheduler
Step #2: #6 sha256:cb853f0b6f8e5648f455b5be89b57707493babad54cdd677be5e8a9a8a3e4637
Step #2: 
Step #2: #6 [2/2] COPY kube-apiserver /usr/local/bin/kube-apiserver
Step #2: #6 sha256:f5eb82ee33d412baf8d16254b05e7a3cdf6e35e0f2e5f0b0fbecf76a8212b213
Step #2: 
Step #2: #6 [2/2] COPY kube-controller-manager /usr/local/bin/kube-controller-manager
Step #2: #6 sha256:842097f21aca0960660e555ae39c09590ee7aa12c5083244d43207ff23ed7029
Step #2: #4 sha256:a3590260a509ebfb98c639d22f55faa45de1287cdb2516fe5bba997d3c9fb578 19.86MB / 25.72MB 1.2s
Step #2: #4 sha256:a3590260a509ebfb98c639d22f55faa45de1287cdb2516fe5bba997d3c9fb578 25.72MB / 25.72MB 1.3s
Step #2: #4 extracting sha256:376d61ddc98ab78b241628e7663b118ecccbc700046029c73e2a333219d21bce 0.9s done
Step #2: #4 sha256:a3590260a509ebfb98c639d22f55faa45de1287cdb2516fe5bba997d3c9fb578 25.72MB / 25.72MB 1.6s done
Step #2: #4 extracting sha256:a3590260a509ebfb98c639d22f55faa45de1287cdb2516fe5bba997d3c9fb578
Step #2: #4 ...
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:0895dd3065e382588aeba84051e89ae8271ec0dc1a893f1aec0044c8d85bfb7e
Step #2: #5 transferring context: 168.08MB 1.5s done
Step #2: #5 DONE 2.4s
Step #2: 
Step #2: #4 [1/8] FROM docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350
Step #2: #4 sha256:b8a6234994ebf3acbd599cfb04d540a2e95b011f85bab1d2d18bfde7458e5951
Step #2: #4 extracting sha256:0ea90407440f407bf09809314aa15cdae15c80cf69ae730af49d75384c35e425
Step #2: #4 extracting sha256:0ea90407440f407bf09809314aa15cdae15c80cf69ae730af49d75384c35e425 0.3s done
Step #2: #4 extracting sha256:a3590260a509ebfb98c639d22f55faa45de1287cdb2516fe5bba997d3c9fb578 1.4s done
Step #2: #4 extracting sha256:bd8432a91c71af90df65ae153260f6fed26d7cd8fd2204da40d9ee140bfe081b
Step #2: #4 extracting sha256:bd8432a91c71af90df65ae153260f6fed26d7cd8fd2204da40d9ee140bfe081b 0.1s done
Step #2: #4 DONE 4.3s
Step #2: 
Step #2: #6 [2/8] COPY ginkgo /usr/local/bin/
Step #2: #6 sha256:fb494f0c3a4650defbdaca5e370305fdfc2849b5e4e3fd57bbd2782ec4e91329
Step #2: #4 extracting sha256:324bff1ba43c9fd3245497b39a42ff88ab08eacc61719c4c21c5af0626175f29 done
Step #2: #4 extracting sha256:7474d8eb55e498e9561206e7a8c10e7412d6692341daa29e969e40f6b507291d
Step #2: #4 extracting sha256:7474d8eb55e498e9561206e7a8c10e7412d6692341daa29e969e40f6b507291d done
Step #2: #4 extracting sha256:ce1ed7d14c9093af116b2d5177d7f1553fb01ad59504b8050424614254274030 done
Step #2: #6 DONE 3.4s
Step #2: #6 DONE 6.7s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #6 DONE 6.8s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #6 DONE 6.8s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #4 DONE 8.4s
Step #2: 
Step #2: #6 [2/2] COPY kube-proxy /usr/local/bin/kube-proxy
Step #2: #6 sha256:48c418450057206e41d6c3be363f4abbae31427cf9c41c7abf1fb0db07da9e4c
Step #2: 
Step #2: #7 [3/8] COPY e2e.test /usr/local/bin/
Step #2: #7 sha256:52d7585df45b27b664184d1cb05ffa87d97e31ccd9f869a13e2f6f6916031774
Step #2: #7 exporting layers 0.7s done
Step #2: #7 writing image sha256:ffd7f718e5a409ce5b126e185d2cb866fc7e3422ca49752c60d600105c51aa49 done
Step #2: #7 naming to k8s.gcr.io/kube-scheduler-s390x:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.7s
Step #2: #7 exporting layers 0.7s done
Step #2: #7 writing image sha256:8017105248c71d682b2e5f06202e5a797da1fd165f736c8e475cb635866323b0 done
Step #2: #7 naming to k8s.gcr.io/kube-controller-manager-s390x:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.7s
Step #2: #7 exporting layers 0.6s done
Step #2: #7 writing image sha256:cbbc0cfb9eaffb18aa3c62a9d6d7f397204ef8bff9916c550c7e0f5116cbb024
Step #2: #7 writing image sha256:cbbc0cfb9eaffb18aa3c62a9d6d7f397204ef8bff9916c550c7e0f5116cbb024 done
Step #2: #7 naming to k8s.gcr.io/kube-apiserver-s390x:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.7s
Step #2: #6 DONE 0.6s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #7 DONE 0.6s
Step #2: 
Step #2: #8 [4/8] COPY kubectl /usr/local/bin/
Step #2: #8 sha256:dd9a5743717d9a3e89fd15eb15dc8e940a44011a720ab31df4c43a7877550027
Step #2: #8 DONE 0.2s
Step #2: 
Step #2: #9 [5/8] COPY run_e2e.sh /run_e2e.sh
Step #2: #9 sha256:a3c8846258a206741089dfffbcad22a67ee013e2cd07e6c49893d35c74be7973
Step #2: #7 exporting layers 0.2s done
Step #2: #7 writing image sha256:831f48a858d0982e2d02fbb0b6f519c58f594b4edb368931877d78f1e67b659b 0.0s done
Step #2: #7 naming to k8s.gcr.io/kube-proxy-s390x:v1.21.0-alpha.3
Step #2: #7 naming to k8s.gcr.io/kube-proxy-s390x:v1.21.0-alpha.3 0.1s done
Step #2: #7 DONE 0.4s
Step #2: #9 DONE 0.2s
Step #2: 
Step #2: #10 [6/8] COPY gorunner /gorunner
Step #2: #10 sha256:dd1802ef0405c5e03ab788dc5fc228208a80e9f3aca6026a90a6f3539414683c
Step #2: #10 DONE 1.5s
Step #2: 
Step #2: #11 [7/8] COPY cluster /kubernetes/cluster
Step #2: #11 sha256:20b32c7768707540a67664ba7ffece41d876a22d771b4fa0ac28035a1a7ac790
Step #2: +++ [0129 14:33:32] Deleting docker image k8s.gcr.io/kube-scheduler-s390x:v1.21.0-alpha.3
Step #2: +++ [0129 14:33:32] Building tarball: node linux-ppc64le
Step #2: +++ [0129 14:33:32] Deleting docker image k8s.gcr.io/kube-apiserver-s390x:v1.21.0-alpha.3
Step #2: +++ [0129 14:33:32] Deleting docker image k8s.gcr.io/kube-controller-manager-s390x:v1.21.0-alpha.3
Step #2: +++ [0129 14:33:32] Deleting docker image k8s.gcr.io/kube-proxy-s390x:v1.21.0-alpha.3
Step #2: #11 DONE 2.7s
Step #2: 
Step #2: #12 [8/8] WORKDIR /usr/local/bin
Step #2: #12 sha256:a467f944dd5eacb80f58d7dc2bd62df06936509b5ec2b2c9281b2648ac124b22
Step #2: #12 DONE 0.0s
Step #2: 
Step #2: #13 exporting to image
Step #2: #13 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #13 exporting layers
Step #2: #13 exporting layers 0.7s done
Step #2: #13 writing image sha256:ab7cacb0b2b3066a1d781063e8f330abae31d28eae1350790a3b68c62ad816e0 done
Step #2: #13 naming to k8s.gcr.io/conformance-s390x:v1.21.0-alpha.3 done
Step #2: #13 DONE 0.7s
Step #2: +++ [0129 14:33:38] Deleting conformance image k8s.gcr.io/conformance-s390x:v1.21.0-alpha.3
Step #2: +++ [0129 14:33:39] Docker builds done
Step #2: +++ [0129 14:33:39] Building images: linux-ppc64le
Step #2: +++ [0129 14:33:39] Starting docker build for image: kube-apiserver-ppc64le
Step #2: +++ [0129 14:33:39] Starting docker build for image: kube-controller-manager-ppc64le
Step #2: +++ [0129 14:33:39] Starting docker build for image: kube-scheduler-ppc64le
Step #2: +++ [0129 14:33:39] Starting docker build for image: kube-proxy-ppc64le
Step #2: +++ [0129 14:33:39] Building conformance image for arch: ppc64le
Step #2: time="2021-01-29T14:33:39Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:33:39Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:33:39Z" level=warning msg="quiet currently not implemented"
Step #2: time="2021-01-29T14:33:39Z" level=warning msg="quiet currently not implemented"
Step #2: #1 [internal] load .dockerignore
Step #2: #1 sha256:cef40c595ae0077bcada8399fe90d81c5c7e2b2e97debcbd9a5ab8ef03d3765f
Step #2: #1 transferring context: 2B done
Step #2: #1 DONE 0.1s
Step #2: 
Step #2: #2 [internal] load build definition from Dockerfile
Step #2: #2 sha256:1dc5e43076795f03a8c09becf7350e9cfb2d3e49dbaa56e0688cf80b39c07258
Step #2: #2 transferring dockerfile: 137B done
Step #2: #2 DONE 0.1s
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0
Step #2: #3 sha256:dbc3c9916ec3869f06629e9cfa2fe8bc977a500f07743ff4c2e16fba6d74dc85
Step #2: #2 [internal] load .dockerignore
Step #2: #2 sha256:846c00f56aa6b061b5df54c77fa7e4be53c5f39c4b10c557e7e44cdc1cb3ae0f
Step #2: #2 transferring context: 2B done
Step #2: #2 DONE 0.0s
Step #2: 
Step #2: #1 [internal] load build definition from Dockerfile
Step #2: #1 sha256:479ab34794eabb88296b760efc253aab4b2d5a9f41a172546a25a878d878daef
Step #2: #1 transferring dockerfile: 139B done
Step #2: #1 DONE 0.0s
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:e315e29ac6941c884eb859b2e43152755ae7870a6590399dd5955d85e4c1a7c0
Step #2: #2 [internal] load build definition from Dockerfile
Step #2: #2 sha256:4a270ef24ddbd56e6bd3e2ddae94a866c4d17c5b18cf40b983ddf0a8db7a35af
Step #2: #2 transferring dockerfile: 139B done
Step #2: #2 DONE 0.0s
Step #2: 
Step #2: #1 [internal] load .dockerignore
Step #2: #1 sha256:da39755bb69924d8b99f102f1856fb89cfc01157e821e20a89b8d345b4b35ae5
Step #2: #1 transferring context: 2B done
Step #2: #1 DONE 0.1s
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:e315e29ac6941c884eb859b2e43152755ae7870a6590399dd5955d85e4c1a7c0
Step #2: #2 [internal] load build definition from Dockerfile
Step #2: #2 sha256:90935341b26d83ceccc3025d79e6528d576fec9dd2e3cd02263758f2ecbc9f2c
Step #2: #2 transferring dockerfile: 157B done
Step #2: #2 DONE 0.1s
Step #2: 
Step #2: #1 [internal] load .dockerignore
Step #2: #1 sha256:c98247583ae6122c6de2e7ca96c1f1a688e96b44c4f48cf2f680720dc88f313d
Step #2: #1 transferring context: 2B done
Step #2: #1 DONE 0.1s
Step #2: 
Step #2: #3 [internal] load metadata for k8s.gcr.io/build-image/go-runner:buster-v2.2.4
Step #2: #3 sha256:e315e29ac6941c884eb859b2e43152755ae7870a6590399dd5955d85e4c1a7c0
Step #2: #2 [internal] load build definition from Dockerfile
Step #2: #2 sha256:7eb4d89dbee38d620e1397b8e4b97adfed0c5b6cc62043f4aca913cd979be29d
Step #2: #2 transferring dockerfile:
Step #2: #3 DONE 0.8s
Step #2: #3 DONE 0.8s
Step #2: #3 DONE 0.9s
Step #2: #2 transferring dockerfile: 1.05kB 0.0s done
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:2d36f6d69e8377adeb28adf6baed2a540d31cd8cd470ea252da11f654626d6f8
Step #2: #4 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:2d36f6d69e8377adeb28adf6baed2a540d31cd8cd470ea252da11f654626d6f8
Step #2: #4 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67
Step #2: #4 sha256:2d36f6d69e8377adeb28adf6baed2a540d31cd8cd470ea252da11f654626d6f8
Step #2: #4 resolve k8s.gcr.io/build-image/go-runner:buster-v2.2.4@sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 done
Step #2: #3 DONE 1.0s
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb
Step #2: #4 sha256:e104a173876422f43fb82c7599a3a024a59067a3c951c6251f6c3e18ff55aaa3
Step #2: #4 resolve k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb done
Step #2: #4 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #4 sha256:1e9368e5a81184eb1fd17053b7796f20663724c2046d17ce394b40f5e47affb1 737B / 737B done
Step #2: #4 sha256:bbc11bdd2bdc389b7c1571dcbed6b8c781d6ec3dce39ac2b60cb0dfca48272c5 1.42kB / 1.42kB done
Step #2: #4 sha256:64053d2c7a53c7f17b4629065bb910dcc6491bc63628e6c0ca98a88ad9f27202 0B / 649.10kB 0.2s
Step #2: #4 sha256:1b07de2aa682b8c1e33636b01fc4091e7025ace84c638b29a6f5e5980da9ddee 0B / 655.74kB 0.2s
Step #2: #4 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #4 sha256:1e9368e5a81184eb1fd17053b7796f20663724c2046d17ce394b40f5e47affb1 737B / 737B done
Step #2: #4 sha256:bbc11bdd2bdc389b7c1571dcbed6b8c781d6ec3dce39ac2b60cb0dfca48272c5 1.42kB / 1.42kB done
Step #2: #4 sha256:64053d2c7a53c7f17b4629065bb910dcc6491bc63628e6c0ca98a88ad9f27202 0B / 649.10kB 0.2s
Step #2: #4 sha256:1b07de2aa682b8c1e33636b01fc4091e7025ace84c638b29a6f5e5980da9ddee 0B / 655.74kB 0.2s
Step #2: #2 DONE 0.5s
Step #2: 
Step #2: #1 [internal] load .dockerignore
Step #2: #1 sha256:b6d33d9081b658a05cbd989bb6755290d6bbe2ce0d1a2ebdb3229814d59a4b21
Step #2: #1 transferring context: 2B done
Step #2: #4 sha256:059fd64240ad0fcd0f6eee6a791004747f1b4a5d3c44dd5ca68258bda4555e67 1.69kB / 1.69kB done
Step #2: #4 sha256:1e9368e5a81184eb1fd17053b7796f20663724c2046d17ce394b40f5e47affb1 737B / 737B done
Step #2: #4 sha256:bbc11bdd2bdc389b7c1571dcbed6b8c781d6ec3dce39ac2b60cb0dfca48272c5 1.42kB / 1.42kB done
Step #2: #4 sha256:64053d2c7a53c7f17b4629065bb910dcc6491bc63628e6c0ca98a88ad9f27202 0B / 649.10kB 0.2s
Step #2: #4 sha256:1b07de2aa682b8c1e33636b01fc4091e7025ace84c638b29a6f5e5980da9ddee 0B / 655.74kB 0.2s
Step #2: #4 extracting sha256:64053d2c7a53c7f17b4629065bb910dcc6491bc63628e6c0ca98a88ad9f27202
Step #2: #4 extracting sha256:64053d2c7a53c7f17b4629065bb910dcc6491bc63628e6c0ca98a88ad9f27202
Step #2: #4 extracting sha256:64053d2c7a53c7f17b4629065bb910dcc6491bc63628e6c0ca98a88ad9f27202 0.1s
Step #2: #1 DONE 1.1s
Step #2: #4 extracting sha256:64053d2c7a53c7f17b4629065bb910dcc6491bc63628e6c0ca98a88ad9f27202 0.2s done
Step #2: #4 extracting sha256:64053d2c7a53c7f17b4629065bb910dcc6491bc63628e6c0ca98a88ad9f27202 0.2s done
Step #2: #4 extracting sha256:64053d2c7a53c7f17b4629065bb910dcc6491bc63628e6c0ca98a88ad9f27202 0.2s done
Step #2: #4 sha256:64053d2c7a53c7f17b4629065bb910dcc6491bc63628e6c0ca98a88ad9f27202 649.10kB / 649.10kB 0.6s done
Step #2: #4 sha256:1b07de2aa682b8c1e33636b01fc4091e7025ace84c638b29a6f5e5980da9ddee 655.74kB / 655.74kB 0.6s done
Step #2: #4 sha256:1ab6759df0558aa874cbb86867d9e7a1b5906d4148d00a5ae63a432a4cb0280d 0B / 27.11MB 0.8s
Step #2: #4 sha256:5353e8ec3ba0fa4715222ec04c617cc53a446e3000a4beb35f044588121e2bac 0B / 14.94MB 0.8s
Step #2: #4 sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb 1.70kB / 1.70kB done
Step #2: #4 sha256:8176fdf7798eb94109c2f118e87a552649403fa8069b45d258a89927aa673196 2.69kB / 2.69kB done
Step #2: #4 sha256:0ea0be4dec281b506f6ceef4cb3594cabea8d80e2dc0d93c7eb09d46259dd837 1.57kB / 1.57kB done
Step #2: #4 sha256:fea3b6521b55e5b841a0cffdaaa603db4fc9e14d6a2b7ecfee56b894ea1d0550 0B / 2.11MB 0.8s
Step #2: #4 sha256:64053d2c7a53c7f17b4629065bb910dcc6491bc63628e6c0ca98a88ad9f27202 649.10kB / 649.10kB 0.6s done
Step #2: #4 sha256:1b07de2aa682b8c1e33636b01fc4091e7025ace84c638b29a6f5e5980da9ddee 655.74kB / 655.74kB 0.6s done
Step #2: #4 extracting sha256:1b07de2aa682b8c1e33636b01fc4091e7025ace84c638b29a6f5e5980da9ddee
Step #2: #4 sha256:64053d2c7a53c7f17b4629065bb910dcc6491bc63628e6c0ca98a88ad9f27202 649.10kB / 649.10kB 0.6s done
Step #2: #4 sha256:1b07de2aa682b8c1e33636b01fc4091e7025ace84c638b29a6f5e5980da9ddee 655.74kB / 655.74kB 0.6s done
Step #2: #4 extracting sha256:1b07de2aa682b8c1e33636b01fc4091e7025ace84c638b29a6f5e5980da9ddee
Step #2: 
Step #2: #3 [internal] load metadata for docker.io/library/debian:stable-slim
Step #2: #3 sha256:a2ebc02c2a22ac7706e45a14be1f17ecdcbd4f8676c9fe52ee1130a61a408829
Step #2: #4 extracting sha256:1b07de2aa682b8c1e33636b01fc4091e7025ace84c638b29a6f5e5980da9ddee 0.1s done
Step #2: #4 extracting sha256:1b07de2aa682b8c1e33636b01fc4091e7025ace84c638b29a6f5e5980da9ddee 0.1s done
Step #2: #4 extracting sha256:1b07de2aa682b8c1e33636b01fc4091e7025ace84c638b29a6f5e5980da9ddee 0.1s done
Step #2: #4 DONE 1.3s
Step #2: #4 DONE 1.3s
Step #2: 
Step #2: 
Step #2: #6 [internal] load build context
Step #2: #5 [internal] load build context
Step #2: #5 sha256:55414b9a7b1d6c2bfe0f91f1b3364e376111fb97a7c95db4f35497dc8e8eb40b
Step #2: #6 sha256:53b57ece59c30d8562060a47451a90b35c4c156ab27b9099628584d88ba490eb
Step #2: #6 transferring context: 105.66MB 1.0s done
Step #2: #5 transferring context: 40.12MB 0.6s done
Step #2: #6 DONE 1.2s
Step #2: #5 DONE 1.2s
Step #2: #4 DONE 1.3s
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:add1fb8c7701deb8596303bd442e18186fc13e12068feb56df8d135e8618a4ec
Step #2: #4 sha256:1ab6759df0558aa874cbb86867d9e7a1b5906d4148d00a5ae63a432a4cb0280d 7.80MB / 27.11MB 1.0s
Step #2: #4 sha256:fea3b6521b55e5b841a0cffdaaa603db4fc9e14d6a2b7ecfee56b894ea1d0550 261.00kB / 2.11MB 1.0s
Step #2: 
Step #2: 
Step #2: #6 [2/2] COPY kube-scheduler /usr/local/bin/kube-scheduler
Step #2: #5 [2/2] COPY kube-controller-manager /usr/local/bin/kube-controller-manager
Step #2: #6 sha256:025833b2e5a4b5feb101cbfd5dc688eac3688d7dab205f5a38ab64e4bd875604
Step #2: #5 sha256:d5f48574f7aaa574975739f1d9108a81f497319080e45fc2c9c9c78bb024b9c1
Step #2: #4 sha256:fea3b6521b55e5b841a0cffdaaa603db4fc9e14d6a2b7ecfee56b894ea1d0550 2.11MB / 2.11MB 1.0s done
Step #2: #4 sha256:8ef934435b56b9bd6db26437e97f1eeb05d1d31ab42eb2fc6a37b9a805981d07 0B / 1.03kB 1.1s
Step #2: #4 sha256:1ab6759df0558aa874cbb86867d9e7a1b5906d4148d00a5ae63a432a4cb0280d 27.11MB / 27.11MB 1.5s
Step #2: #4 sha256:5353e8ec3ba0fa4715222ec04c617cc53a446e3000a4beb35f044588121e2bac 14.94MB / 14.94MB 1.5s
Step #2: #4 sha256:8ef934435b56b9bd6db26437e97f1eeb05d1d31ab42eb2fc6a37b9a805981d07 1.03kB / 1.03kB 1.2s done
Step #2: #4 sha256:678693ead7d0b405bd48408cdc9b48b807d94bf68e3b05aeeabe3917ebf379fd 0B / 520B 1.5s
Step #2: #3 DONE 0.8s
Step #2: #4 ...
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:acc794aa41f740afbc47cf31db6955fd6e8832c183e9795e2c3d4baa38f2b2fe
Step #2: #5 transferring context: 36.97MB 0.5s done
Step #2: #5 DONE 1.8s
Step #2: 
Step #2: #4 [1/8] FROM docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350
Step #2: #4 sha256:5df6338bef9c758af5a26b77c75ac9980773a00cb76cab974f7bf67cbcf512c0
Step #2: #4 resolve docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350 done
Step #2: 
Step #2: #4 [1/2] FROM k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0@sha256:abe8cef9e116f2d5ec1175c386e33841ff3386779138b425af876384b0fd7ccb
Step #2: #4 sha256:e104a173876422f43fb82c7599a3a024a59067a3c951c6251f6c3e18ff55aaa3
Step #2: #4 sha256:1ab6759df0558aa874cbb86867d9e7a1b5906d4148d00a5ae63a432a4cb0280d 27.11MB / 27.11MB 1.7s done
Step #2: #4 sha256:5353e8ec3ba0fa4715222ec04c617cc53a446e3000a4beb35f044588121e2bac 14.94MB / 14.94MB 1.6s done
Step #2: #4 sha256:678693ead7d0b405bd48408cdc9b48b807d94bf68e3b05aeeabe3917ebf379fd 520B / 520B 1.7s done
Step #2: #4 extracting sha256:1ab6759df0558aa874cbb86867d9e7a1b5906d4148d00a5ae63a432a4cb0280d
Step #2: #4 sha256:5960416713d42f9ba367c7080ad1906beb8a1830c35cb74d584df4560c165d0d 0B / 549B 1.8s
Step #2: #4 sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350 1.85kB / 1.85kB done
Step #2: #4 sha256:1b60509bd7e6b8722e3681c1275161d08cb4979652a0d0b1d563a5f5693f1d3a 529B / 529B done
Step #2: #4 sha256:cb67ad3b1682d91fbbb6b4550f7c7bd6924c3c98e6609ebff029e118f4525b92 1.47kB / 1.47kB done
Step #2: #4 sha256:e5d1bed0e20399ef2576d4759053693b944aebfe4f69e161ec49293e587b32c5 0B / 30.53MB 0.2s
Step #2: #4 sha256:5960416713d42f9ba367c7080ad1906beb8a1830c35cb74d584df4560c165d0d 549B / 549B 2.1s
Step #2: #5 transferring context: 111.23MB 1.2s done
Step #2: #4 sha256:5960416713d42f9ba367c7080ad1906beb8a1830c35cb74d584df4560c165d0d 549B / 549B 2.1s done
Step #2: #5 DONE 2.6s
Step #2: #4 sha256:e5d1bed0e20399ef2576d4759053693b944aebfe4f69e161ec49293e587b32c5 8.39MB / 30.53MB 0.6s
Step #2: 
Step #2: #6 [2/2] COPY kube-apiserver /usr/local/bin/kube-apiserver
Step #2: #6 sha256:f5efe23228f38cdd37a3ac36470ad0d8593695d74365177a8505e95e498edf9f
Step #2: #4 sha256:e5d1bed0e20399ef2576d4759053693b944aebfe4f69e161ec49293e587b32c5 12.18MB / 30.53MB 0.9s
Step #2: #4 sha256:e5d1bed0e20399ef2576d4759053693b944aebfe4f69e161ec49293e587b32c5 30.53MB / 30.53MB 1.0s
Step #2: #4 extracting sha256:1ab6759df0558aa874cbb86867d9e7a1b5906d4148d00a5ae63a432a4cb0280d 1.1s done
Step #2: #4 sha256:e5d1bed0e20399ef2576d4759053693b944aebfe4f69e161ec49293e587b32c5 30.53MB / 30.53MB 1.2s done
Step #2: #4 extracting sha256:e5d1bed0e20399ef2576d4759053693b944aebfe4f69e161ec49293e587b32c5
Step #2: #4 ...
Step #2: 
Step #2: #5 [internal] load build context
Step #2: #5 sha256:ab3e4a1347b3d3ff708bdcc28030dd72dfe3748d11fb1dea27edcfe9436c1ad6
Step #2: #5 transferring context: 162.91MB 1.4s done
Step #2: #5 DONE 2.4s
Step #2: 
Step #2: #4 [1/8] FROM docker.io/library/debian:stable-slim@sha256:8e75af576baa10409a1cd8e9d1b0b7b8a636b89ae8665b84697c100511a65350
Step #2: #4 sha256:5df6338bef9c758af5a26b77c75ac9980773a00cb76cab974f7bf67cbcf512c0
Step #2: #4 extracting sha256:5353e8ec3ba0fa4715222ec04c617cc53a446e3000a4beb35f044588121e2bac 0.1s
Step #2: #4 extracting sha256:5353e8ec3ba0fa4715222ec04c617cc53a446e3000a4beb35f044588121e2bac 0.5s done
Step #2: #4 extracting sha256:e5d1bed0e20399ef2576d4759053693b944aebfe4f69e161ec49293e587b32c5 1.5s done
Step #2: #4 extracting sha256:fea3b6521b55e5b841a0cffdaaa603db4fc9e14d6a2b7ecfee56b894ea1d0550
Step #2: #4 extracting sha256:fea3b6521b55e5b841a0cffdaaa603db4fc9e14d6a2b7ecfee56b894ea1d0550 0.2s done
Step #2: #4 DONE 4.0s
Step #2: 
Step #2: #6 [2/8] COPY ginkgo /usr/local/bin/
Step #2: #6 sha256:3d74bd37511575cf187fa56e51aeaa02e1151a77e4409600f9b3d1cbf53efd77
Step #2: #4 extracting sha256:8ef934435b56b9bd6db26437e97f1eeb05d1d31ab42eb2fc6a37b9a805981d07
Step #2: #6 DONE 4.8s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #5 DONE 4.8s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #4 extracting sha256:8ef934435b56b9bd6db26437e97f1eeb05d1d31ab42eb2fc6a37b9a805981d07 done
Step #2: #6 DONE 0.2s
Step #2: 
Step #2: #7 [3/8] COPY e2e.test /usr/local/bin/
Step #2: #7 sha256:cc1920cd7f59ff9c6ff2a9851cd721a5ed4ccd286386767ce9cb7c2dae5a9a8b
Step #2: #4 extracting sha256:678693ead7d0b405bd48408cdc9b48b807d94bf68e3b05aeeabe3917ebf379fd done
Step #2: #4 extracting sha256:5960416713d42f9ba367c7080ad1906beb8a1830c35cb74d584df4560c165d0d
Step #2: #7 exporting layers 1.0s done
Step #2: #7 writing image sha256:78b066927011ffa0f9d74ce5ecfbd78ffdbfc843d24aa46b44401d0b63dc3017 0.0s done
Step #2: #7 naming to k8s.gcr.io/kube-scheduler-ppc64le:v1.21.0-alpha.3 done
Step #2: #7 DONE 1.0s
Step #2: #7 exporting layers 1.0s done
Step #2: #7 writing image sha256:cf8df40bc84f1395eb3ddb1d755f5d25d6d0d7a204bc097817689fa3b9db68ef done
Step #2: #7 naming to k8s.gcr.io/kube-controller-manager-ppc64le:v1.21.0-alpha.3 done
Step #2: #7 DONE 1.1s
Step #2: #6 DONE 4.4s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #7 DONE 0.9s
Step #2: 
Step #2: #8 [4/8] COPY kubectl /usr/local/bin/
Step #2: #8 sha256:a35bd91a851a6b9c27f818b0030a6bb76242356d37deb8f80f3eb9711379363f
Step #2: #4 extracting sha256:5960416713d42f9ba367c7080ad1906beb8a1830c35cb74d584df4560c165d0d done
Step #2: #4 DONE 7.2s
Step #2: 
Step #2: #6 [2/2] COPY kube-proxy /usr/local/bin/kube-proxy
Step #2: #6 sha256:c97e2a552da8cf84c7629a957169437bddecca55a40c24419225805e6f55db07
Step #2: #8 DONE 0.3s
Step #2: 
Step #2: #9 [5/8] COPY run_e2e.sh /run_e2e.sh
Step #2: #9 sha256:d7e77333aa1c6d9fb47a721d087a802844ad704296ec6da7113c9e6e59ee7903
Step #2: #6 DONE 0.3s
Step #2: 
Step #2: #7 exporting to image
Step #2: #7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #7 exporting layers
Step #2: #9 DONE 0.3s
Step #2: 
Step #2: #10 [6/8] COPY gorunner /gorunner
Step #2: #10 sha256:955954106835a80a36bd7b9a08ebf5796cabd2e7484f02e223c52c1c88be4cca
Step #2: #7 exporting layers 0.8s done
Step #2: #7 writing image sha256:8b8c8ad41f3b78b5d6efdd52dc765e85a92ec2be60a86ddd2fba33e3040bfa32 done
Step #2: #7 naming to k8s.gcr.io/kube-apiserver-ppc64le:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.8s
Step #2: #10 DONE 0.2s
Step #2: 
Step #2: #11 [7/8] COPY cluster /kubernetes/cluster
Step #2: #11 sha256:ec3a9cc444c8d2ba4835b450b01f3dd8acf6256774c5b6fa0bd3736c888b97e5
Step #2: +++ [0129 14:33:48] Deleting docker image k8s.gcr.io/kube-scheduler-ppc64le:v1.21.0-alpha.3
Step #2: #11 DONE 0.3s
Step #2: #7 exporting layers 0.5s done
Step #2: #7 writing image sha256:103259ac601013e6f7667bbf54d383fb2a04fb9af787aa0c36db51d5f99d0177 done
Step #2: #7 naming to k8s.gcr.io/kube-proxy-ppc64le:v1.21.0-alpha.3 done
Step #2: #7 DONE 0.6s
Step #2: 
Step #2: #12 [8/8] WORKDIR /usr/local/bin
Step #2: #12 sha256:35a62f6ab95f27f1bd97c7206765e09751f598e48b3e2288c6faf857916ced38
Step #2: +++ [0129 14:33:48] Building tarball: node windows-amd64
Step #2: #12 DONE 0.6s
Step #2: 
Step #2: #13 exporting to image
Step #2: #13 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
Step #2: #13 exporting layers
Step #2: +++ [0129 14:33:49] Deleting docker image k8s.gcr.io/kube-controller-manager-ppc64le:v1.21.0-alpha.3
Step #2: #13 exporting layers 2.6s done
Step #2: #13 writing image sha256:069fa3ebe3533ff61c03c70c57d9295af0d01d8e7d1a9104b0f54abf9c805967 done
Step #2: #13 naming to k8s.gcr.io/conformance-ppc64le:v1.21.0-alpha.3 done
Step #2: #13 DONE 2.6s
Step #2: +++ [0129 14:33:52] Deleting docker image k8s.gcr.io/kube-apiserver-ppc64le:v1.21.0-alpha.3
Step #2: +++ [0129 14:33:52] Deleting docker image k8s.gcr.io/kube-proxy-ppc64le:v1.21.0-alpha.3
Step #2: +++ [0129 14:33:56] Deleting conformance image k8s.gcr.io/conformance-ppc64le:v1.21.0-alpha.3
```

Can you share some more insights which error cases we might face?

Reply:
@saschagrunert we were previously hitting a bug with the base image not being cached on registry mirror, but it is possible that issue has been addressed on GCR side (https://github.com/kubernetes/kubernetes/pull/92042) :+1:

Reply:
this is also addressed on the buildx side going forward (should fallback to the non mirror endpoint like docker build does), thought it may be some time before we're using the patched version.